# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
#   <workisgreat@yandex.com>, 2012.
#: ../../client/solo/commands/query.py:238
msgid ""
msgstr ""
"Project-Id-Version: Sabayon Entropy Client\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-11-06 22:12+0100\n"
"PO-Revision-Date: 2012-11-03 23:59+0000\n"
"Last-Translator: lxnay <lxnay@sabayon.org>\n"
"Language-Team: Russian (http://www.transifex.com/projects/p/sabayon-entropy/"
"language/ru/)\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: ../../lib/entropy/db/sqlite.py:1676
msgid "Exporting database table"
msgstr "Экспорт таблицы базы данных"

#: ../../lib/entropy/db/sqlite.py:1716
msgid "Database Export complete."
msgstr "Экспорт базы данных завершен."

#: ../../lib/entropy/db/sqlite.py:2094 ../../lib/entropy/db/sqlite.py:2198
#: ../../lib/entropy/db/sqlite.py:2309 ../../lib/entropy/db/sqlite.py:2322
#: ../../lib/entropy/db/sql.py:5405 ../../lib/entropy/db/sql.py:5418
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2625
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2763
#: ../../lib/entropy/client/interfaces/client.py:949
msgid "ATTENTION"
msgstr "ВНИМАНИЕ"

#: ../../lib/entropy/db/sqlite.py:2096 ../../lib/entropy/db/sqlite.py:2200
msgid "updating repository metadata layout, please wait!"
msgstr ""

#: ../../lib/entropy/db/sqlite.py:2310
msgid "generating provided_libs metadata, please wait!"
msgstr "генерируются метаданные provided_libs, пожалуйста, подождите!"

#: ../../lib/entropy/db/sql.py:5406
msgid "Spm error occured"
msgstr ""

#: ../../lib/entropy/db/sql.py:5419
msgid "Spm Unique Identifier not found for"
msgstr ""

#: ../../lib/entropy/db/skel.py:1680 ../../lib/entropy/db/skel.py:2028
#: ../../lib/entropy/client/interfaces/trigger.py:522
#: ../../lib/entropy/client/interfaces/trigger.py:532
#: ../../lib/entropy/client/interfaces/trigger.py:542
#: ../../lib/entropy/client/interfaces/trigger.py:552
#: ../../lib/entropy/client/interfaces/trigger.py:562
msgid "SPM"
msgstr "SPM"

#: ../../lib/entropy/db/skel.py:1681
msgid "Running packages metadata update"
msgstr "Ведется обновление метаданных пакетов"

#: ../../lib/entropy/db/skel.py:1682
msgid "it could take a while"
msgstr "это может занять время"

#: ../../lib/entropy/db/skel.py:1702 ../../lib/entropy/db/skel.py:1728
#: ../../lib/entropy/db/skel.py:1751
msgid "Entropy"
msgstr ""

#: ../../lib/entropy/db/skel.py:1703
msgid "action"
msgstr "действие"

#: ../../lib/entropy/db/skel.py:1729
msgid "package move actions complete"
msgstr "перемещение пакета завершено"

#: ../../lib/entropy/db/skel.py:1743
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2605
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2800
msgid "WARNING"
msgstr "ОСТОРОЖНО"

#: ../../lib/entropy/db/skel.py:1744
msgid "Cannot run SPM cleanup, error"
msgstr "Невозможно запустить очистку SPM, ошибка"

#: ../../lib/entropy/db/skel.py:1752
msgid "package moves completed successfully"
msgstr "перемещения пакета успешно завершены"

#: ../../lib/entropy/db/skel.py:2029
msgid "Moving old entry"
msgstr "Перемещение старой записи"

#: ../../lib/entropy/db/skel.py:2031
msgid "to"
msgstr "на"

#: ../../lib/entropy/db/skel.py:3800
msgid "Syncing current database"
msgstr "Синхронизация текущей базы данных"

#: ../../lib/entropy/db/skel.py:3801
#: ../../lib/entropy/client/interfaces/db.py:299
msgid "please wait"
msgstr "пожалуйста, подождите"

#: ../../lib/entropy/db/skel.py:3816
msgid "Removing entry"
msgstr "Удаление записи"

#: ../../lib/entropy/db/skel.py:3835
msgid "Adding entry"
msgstr "Добавление записи"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:248
msgid "Accessibility"
msgstr "Специальные возможности"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:250
msgid "Accessibility applications"
msgstr "Приложения специальных возможностей"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:254
msgid "Office"
msgstr "Офис"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:255
msgid "Applications used in office environments"
msgstr "Приложения, используемые в офисной среде"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:261
msgid "Development"
msgstr "Разработка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:262
msgid "Applications or system libraries"
msgstr "Приложения или системные библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:266
msgid "System"
msgstr "Система"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:267
msgid "System applications or libraries"
msgstr "Системные приложения или библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:271
msgid "Games"
msgstr "Игры"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:272
msgid "Games, enjoy your spare time"
msgstr "Игры, наслаждайтесь своим отдыхом"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:276
msgid "GNOME Desktop"
msgstr "Среда GNOME"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:278
msgid "Applications and libraries for the GNOME Desktop"
msgstr "Приложения и библиотеки для среды GNOME"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:282
msgid "KDE Desktop"
msgstr "Среда KDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:284
msgid "Applications and libraries for the KDE Desktop"
msgstr "Приложения и библиотеки для среды KDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:288
msgid "XFCE Desktop"
msgstr "Среда XFCE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:290
msgid "Applications and libraries for the XFCE Desktop"
msgstr "Приложения и библиотеки для среды XFCE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:294
msgid "LXDE Desktop"
msgstr "Среда LXDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:296
msgid "Applications and libraries for the LXDE Desktop"
msgstr "Приложения и библиотеки для среды LXDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:300
msgid "Multimedia"
msgstr "Мультимедиа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:302
msgid "Applications and libraries for Multimedia"
msgstr "Приложения и библиотеки для мультимедиа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:306
msgid "Networking"
msgstr "Сеть"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:308
msgid "Applications and libraries for Networking"
msgstr "Приложения и библиотеки для подключения к сети"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:312
msgid "Science"
msgstr "Наука"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:314
msgid "Scientific applications and libraries"
msgstr "Научные приложения и библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:318
msgid "Security"
msgstr "Безопасность"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:320
msgid "Security oriented applications"
msgstr "Приложения раздела \"Безопасность\""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:324
msgid "X11"
msgstr "X11"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:326
msgid "Applications and libraries for X11"
msgstr "Приложения и библиотеки для X11"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:610
msgid "Reloading Portage modules"
msgstr "Модули Portage перезагружаются"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:1098
msgid "error"
msgstr "ошибка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:1100
#: ../../lib/entropy/client/interfaces/package.py:693
#: ../../lib/entropy/client/interfaces/package.py:1374
msgid "not found"
msgstr "не найдено"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2290
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2364
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2456
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2528
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3179
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4821
#: ../../lib/entropy/client/interfaces/trigger.py:280
msgid "QA"
msgstr "QA"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2291
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2365
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2457
msgid "Cannot run Source Package Manager trigger for"
msgstr "Невозможно запустить триггер Источника Пакетного Менеджера для"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2300
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2374
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2466
#: ../../lib/entropy/client/interfaces/trigger.py:283
msgid "Please report it"
msgstr "Пожалуйста, сообщите об этом"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2301
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2375
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2467
msgid "Attach this"
msgstr "Вложить это"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2529
msgid "Cannot run SPM configure phase for"
msgstr "Невозможно запустить фазу конфигурации SPM для"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2533
#: ../../lib/entropy/client/interfaces/package.py:3492
#: ../../lib/entropy/client/interfaces/package.py:3529
#: ../../lib/entropy/client/interfaces/package.py:3656
#: ../../lib/entropy/client/interfaces/package.py:3663
#: ../../lib/entropy/client/interfaces/package.py:3719
#: ../../lib/entropy/client/interfaces/package.py:3747
#: ../../lib/entropy/client/interfaces/package.py:3973
#: ../../lib/entropy/client/interfaces/package.py:3993
#: ../../lib/entropy/client/interfaces/trigger.py:74
#: ../../lib/entropy/security.py:659 ../../client/solo/commands/rescue.py:198
msgid "Error"
msgstr "Ошибка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2584
msgid "package not available on system"
msgstr "пакет недоступен в системе"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2593
msgid "repackaging"
msgstr "перепаковка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2606
msgid "Cannot complete quickpkg for atom"
msgstr "Невозможно завершить quickpkg для атома"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2608
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2803
msgid "do it manually"
msgstr "сделайте это вручную"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2626
msgid "package files rebuild did not run properly"
msgstr "пересборка файлов пакета не запущена корректно"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2627
msgid "Please update packages manually"
msgstr "Будьте добры обновить пакеты вручную"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2764
msgid "forcing package updates"
msgstr "принуждение к обновлению пакетов"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2765
msgid "Syncing with"
msgstr "Синхронизация с"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2801
msgid "Cannot complete quickpkg for atoms"
msgstr "Невозможно завершить quickpkg для атомов"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3180
msgid "Cannot update Portage package metadata"
msgstr ""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3206
msgid "SPM uid update error"
msgstr "Ошибка при обновлении SPM uid"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3275
msgid "Cannot update SPM installed pkgs file"
msgstr "Невозможно обновить файл установленных пакетов SPM"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3368
msgid "Portage world file is corrupted"
msgstr "Файл Portage world поврежден"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3422
#: ../../lib/entropy/client/interfaces/db.py:387
#: ../../lib/entropy/client/interfaces/db.py:490
#: ../../lib/entropy/client/interfaces/db.py:596
#: ../../lib/entropy/client/interfaces/db.py:1905 ../../lib/entropy/qa.py:802
#: ../../client/solo/commands/install.py:456
#: ../../client/solo/commands/install.py:459
#: ../../client/solo/commands/install.py:466
#: ../../client/solo/commands/install.py:469
#: ../../client/solo/commands/rescue.py:305
#: ../../client/solo/commands/rescue.py:316 ../../services/kernel-switcher:140
msgid "Attention"
msgstr "Внимание"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3423
msgid "preserved libraries have been found on system"
msgstr "в системе найдены устаревшие библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3558
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3708
msgid "does not exist"
msgstr "не существует"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3558
msgid "Overwriting"
msgstr "Перезапись"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3604
msgid "variable differs"
msgstr "всевозможные различия"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3604
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3693
msgid "Updating"
msgstr "Обновление"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3619
msgid "updating critical variables"
msgstr "Обновление критических переменных"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3693
msgid "differs"
msgstr "различия"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3708
msgid "Reverting"
msgstr "Возвращение"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3723
msgid "Skipping configuration files update, you are not root."
msgstr "Пропуск обновления файлов конфигурации, Вы не root."

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3984
msgid "Error calculating dependencies"
msgstr "Ошибка при подсчёте зависимостей"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4543
msgid "not supported"
msgstr "не поддерживается"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4544
msgid "Probably Portage API has changed"
msgstr "Возможно, был изменен API портеджа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4665
msgid "IOError while reading"
msgstr "Ошибка Ввода-Вывода при чтении"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4822
msgid "illegal Entropy package tag in ebuild"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:61
msgid "Unknown documents"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:62
msgid "Comments"
msgstr "Комментарии"

#. BBCODE_TYPE_ID: _('BBcode Documents'),
#: ../../lib/entropy/client/services/interfaces.py:64
msgid "Images/Screenshots"
msgstr "Изображения/скриншоты"

#: ../../lib/entropy/client/services/interfaces.py:65
msgid "Generic Files"
msgstr "Общие файлы"

#: ../../lib/entropy/client/services/interfaces.py:66
msgid "Videos"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:67
msgid "Icons"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:70
msgid "Unknown document"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:71
msgid "Comment"
msgstr "Комментарий"

#. BBCODE_TYPE_ID: _('BBcode Document'),
#: ../../lib/entropy/client/services/interfaces.py:73
msgid "Image/Screenshot"
msgstr "Изображение/скриншот"

#: ../../lib/entropy/client/services/interfaces.py:74
msgid "Generic File"
msgstr "Общий файл"

#: ../../lib/entropy/client/services/interfaces.py:75
msgid "Video"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:76
msgid "Icon"
msgstr ""

#: ../../lib/entropy/client/misc.py:213 ../../lib/entropy/client/misc.py:579
msgid "Automerging file"
msgstr "Автообъединение файла"

#: ../../lib/entropy/client/misc.py:247
msgid "Found update"
msgstr ""

#: ../../lib/entropy/client/misc.py:595
msgid "System Error"
msgstr "Системная ошибка"

#: ../../lib/entropy/client/misc.py:596
msgid "Cannot automerge file"
msgstr "Невозможно автоматически объединить файл"

#: ../../lib/entropy/client/interfaces/sets.py:238
msgid "Name already taken"
msgstr "Имя уже установлено"

#: ../../lib/entropy/client/interfaces/sets.py:247
msgid "Cannot remove the old element"
msgstr "Невозможно удалить старый элемент"

#: ../../lib/entropy/client/interfaces/sets.py:249
msgid "Cannot create the element"
msgstr "Невозможно создать элемент"

#: ../../lib/entropy/client/interfaces/sets.py:276
msgid "cannot start with"
msgstr "невозможно стартовать с"

#: ../../lib/entropy/client/interfaces/sets.py:281
msgid "Already removed"
msgstr "Уже удалено"

#: ../../lib/entropy/client/interfaces/sets.py:285
msgid "Not defined by user"
msgstr "Не указано пользователем"

#: ../../lib/entropy/client/interfaces/sets.py:293
msgid "Set not found or unable to remove"
msgstr "Набор не найден или невозможно удалить"

#: ../../lib/entropy/client/interfaces/cache.py:106
#: ../../lib/entropy/client/interfaces/methods.py:100
#: ../../lib/entropy/client/interfaces/methods.py:125
#: ../../client/solo/commands/upgrade.py:254
msgid "Repository"
msgstr "Репозиторий"

#: ../../lib/entropy/client/interfaces/cache.py:107
#: ../../lib/entropy/client/interfaces/methods.py:126
msgid "is corrupted"
msgstr "испорчен"

#: ../../lib/entropy/client/interfaces/cache.py:108
msgid "Cannot calculate the checksum"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:81
msgid "Please update your repositories now in order to remove this message!"
msgstr ""
"Пожалуйста, обновите ваши репозитории сейчас, чтобы убрать это сообщение!"

#: ../../lib/entropy/client/interfaces/methods.py:101
#: ../../client/solo/commands/pkg.py:197
#: ../../client/solo/commands/mask.py:101
msgid "is not available"
msgstr "недоступен"

#: ../../lib/entropy/client/interfaces/methods.py:101
#: ../../lib/entropy/client/interfaces/methods.py:126
msgid "Cannot validate"
msgstr "Невозможно подтвердить"

#: ../../lib/entropy/client/interfaces/methods.py:275
msgid "bad repository id specified"
msgstr "выбран неправильный ID репозитория"

#: ../../lib/entropy/client/interfaces/methods.py:300
#, python-format
msgid "Repository %s hasn't been downloaded yet."
msgstr "Репозиторий %s ещё не скачан."

#: ../../lib/entropy/client/interfaces/methods.py:1089
msgid "System database not found or corrupted"
msgstr "База данных системы не найдена или испорчена"

#: ../../lib/entropy/client/interfaces/methods.py:1090
msgid "running in safe mode using temporary, empty repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1270
msgid "Backing up repository to"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1290
msgid "Repository backed up successfully"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1299
#: ../../lib/entropy/client/interfaces/methods.py:1357
#: ../../lib/entropy/const.py:683
msgid "All fine"
msgstr "Всё замечательно"

#: ../../lib/entropy/client/interfaces/methods.py:1318
msgid "Restoring backed up repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1336
msgid "Unable to unpack"
msgstr "Невозможно распаковать"

#: ../../lib/entropy/client/interfaces/methods.py:1346
msgid "Repository restored successfully"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1469
#: ../../lib/entropy/client/interfaces/package.py:4100
msgid "Removing"
msgstr "Удаление"

#: ../../lib/entropy/client/interfaces/methods.py:1916
msgid "Resources unlocked, let's go!"
msgstr "Ресурсы освобождены. Поехали!"

#: ../../lib/entropy/client/interfaces/methods.py:1924
msgid "Resources still locked, giving up!"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1932
msgid "Resources locked, sleeping..."
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2126
msgid "Checking speed of"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2157
msgid "Mirror speed"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2671
msgid "not a valid method"
msgstr "неправильный метод"

#: ../../lib/entropy/client/interfaces/db.py:297
msgid "Injecting downloaded dump"
msgstr "Добавляется объём загруженного"

#: ../../lib/entropy/client/interfaces/db.py:388
msgid "repository is not available at the following URL:"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:399
msgid "Looking for an alternate route using package mirrors"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:411
#, fuzzy
msgid "Checking repository URL"
msgstr "Пропуск репозитория"

#: ../../lib/entropy/client/interfaces/db.py:432
#, fuzzy
msgid "Found repository at URL"
msgstr "Некорректный репозиторий"

#: ../../lib/entropy/client/interfaces/db.py:444
msgid "Unable to find alternate repository mirrors. Sorry."
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:460
#: ../../client/solo/commands/status.py:153
msgid "Repository URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:468
msgid "Repository local path"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:476
msgid "Repository API"
msgstr "API репозитория"

#: ../../lib/entropy/client/interfaces/db.py:491
msgid "Repository is being updated"
msgstr "Репозиторий обновляется"

#: ../../lib/entropy/client/interfaces/db.py:492
msgid "Try again in a few minutes"
msgstr "Попробуйте снова через несколько минут"

#. starting to download
#: ../../lib/entropy/client/interfaces/db.py:553
msgid "Downloading repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:597
msgid "unable to download the repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:617
msgid "Downloading checksum"
msgstr "Загрузка контрольной суммы"

#: ../../lib/entropy/client/interfaces/db.py:642
msgid "Cannot fetch checksum"
msgstr "Невозможно загрузить контрольную сумму"

#: ../../lib/entropy/client/interfaces/db.py:643
#: ../../lib/entropy/client/interfaces/db.py:1219
msgid "Cannot verify repository integrity"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:734
msgid "Unpacking database to"
msgstr "Распаковка базы данных в"

#: ../../lib/entropy/client/interfaces/db.py:751
msgid "Cannot unpack compressed package"
msgstr "Невозможно распаковать сжатый пакет"

#: ../../lib/entropy/client/interfaces/db.py:752
msgid "Skipping repository"
msgstr "Пропуск репозитория"

#: ../../lib/entropy/client/interfaces/db.py:786
msgid "Repository is invalid"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:810
msgid "Indexing Repository metadata"
msgstr "Индексирование метаинформации репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1026
msgid "Downloading repository metafile"
msgstr "Скачивание метафайла репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1036
msgid "Downloading GPG signature of repository metafile"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1046
msgid "Downloading Notice Board"
msgstr "Загрузка доски уведомлений"

#: ../../lib/entropy/client/interfaces/db.py:1072
msgid "unpacked meta file"
msgstr "распакованный файл meta"

#: ../../lib/entropy/client/interfaces/db.py:1078
msgid "removed meta file"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1095
msgid "not available, it's ok"
msgstr "недоступно, но это нормально"

#: ../../lib/entropy/client/interfaces/db.py:1099
msgid "not available, not very ok!"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1112
msgid "available, w00t!"
msgstr "ура, доступно!"

#: ../../lib/entropy/client/interfaces/db.py:1130
msgid "cannot be unpacked, not very ok!"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1181
msgid "Repository revision"
msgstr "Версия репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1204
msgid "Checking downloaded repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1218
msgid "Cannot open digest"
msgstr "Невозможно открыть обзор"

#: ../../lib/entropy/client/interfaces/db.py:1229
#: ../../lib/entropy/client/interfaces/db.py:1240
msgid "Downloaded repository status"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1230
msgid "OK"
msgstr "OK"

#: ../../lib/entropy/client/interfaces/db.py:1241
msgid "ERROR"
msgstr "ОШИБКА"

#: ../../lib/entropy/client/interfaces/db.py:1250
msgid "An error occured while checking repository integrity"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1251
msgid "Giving up"
msgstr "Прекращаю"

#: ../../lib/entropy/client/interfaces/db.py:1273
#: ../../lib/entropy/security.py:284
msgid "Make sure to verify the imported key and set an appropriate trust level"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1292
msgid "This repository suports GPG-signed packages"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1299
msgid "you may want to install GnuPG to take advantage of this feature"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1333
#: ../../lib/entropy/security.py:323
msgid "GPG key changed for"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1353
#: ../../lib/entropy/security.py:342
msgid "GPG key already installed for"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1366
msgid "GPG key EXPIRED for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1378
msgid "Installing GPG key for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1396
#: ../../lib/entropy/client/interfaces/db.py:1428
#: ../../lib/entropy/security.py:380
msgid "Error during GPG key installation"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1406
msgid "GPG key seems already installed but not properly recorded, resetting"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1440
msgid "Successfully installed GPG key for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1449
#: ../../lib/entropy/security.py:400
msgid "Fingerprint"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1489
msgid "Verifying GPG signature of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1503
msgid "Verified GPG signature of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1513
#: ../../lib/entropy/security.py:425
msgid "Error during GPG verification of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1522
#: ../../lib/entropy/security.py:434
msgid "It could mean a potential security risk"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1577
msgid "Web Service"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1578
msgid "skipping differential sync"
msgstr "пропуск различной синхронизации"

#: ../../lib/entropy/client/interfaces/db.py:1579
msgid "threshold"
msgstr "порог"

#: ../../lib/entropy/client/interfaces/db.py:1613
msgid "Web Service communication error"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1633
msgid "Web Service data error"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1649
msgid "Local status"
msgstr "Локальный статус"

#: ../../lib/entropy/client/interfaces/db.py:1681
msgid "Fetching segments"
msgstr "Загрузка сегментов"

#: ../../lib/entropy/client/interfaces/db.py:1717
#: ../../lib/entropy/client/interfaces/db.py:1736
#: ../../lib/entropy/client/interfaces/db.py:1753
msgid "Web Service status"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1718
msgid "cannot fetch repository metadata"
msgstr "невозможно загрузить метаинформацию репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1737
msgid "cannot update treeupdates data"
msgstr "невозможно обновить treeupdates-информацию"

#: ../../lib/entropy/client/interfaces/db.py:1754
msgid "cannot update package sets data"
msgstr "невозможно обновить информацию по наборам пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1770
msgid "Fetch error on segment while adding"
msgstr "Ошибка загрузки сегмента при добавлении"

#: ../../lib/entropy/client/interfaces/db.py:1796
msgid "repository error while adding packages"
msgstr "Ошибка репозитория при добавлении пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1819
msgid "repository error while removing packages"
msgstr "ошибка репозитория при удалении пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1835
msgid "Repository checksum doesn't match remote."
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1838
msgid "local"
msgstr "локально"

#: ../../lib/entropy/client/interfaces/db.py:1843
msgid "remote"
msgstr "отдаленное"

#: ../../lib/entropy/client/interfaces/db.py:1906
msgid "repository is already up to date"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:2110
msgid "Configuration files update error, not critical, continuing"
msgstr "Ошибка при обновлении файлов конфигурации, некритично, продолжаем"

#: ../../lib/entropy/client/interfaces/package.py:373
msgid "Already prepared"
msgstr "Уже подготовлено"

#: ../../lib/entropy/client/interfaces/package.py:378
msgid "Not yet prepared"
msgstr "Ещё не подготовлено"

#: ../../lib/entropy/client/interfaces/package.py:610
#: ../../lib/entropy/client/interfaces/package.py:1281
msgid "Mirror"
msgstr "Зеркало"

#: ../../lib/entropy/client/interfaces/package.py:612
#: ../../lib/entropy/client/interfaces/package.py:1283
msgid "maximum failure threshold reached"
msgstr "Максимальный порог ошибок достигнут"

#: ../../lib/entropy/client/interfaces/package.py:657
msgid "success"
msgstr "успех"

#: ../../lib/entropy/client/interfaces/package.py:666
msgid "Aggregated transfer rate"
msgstr "Аггрегированная скорость передачи"

#: ../../lib/entropy/client/interfaces/package.py:669
#: ../../lib/entropy/client/interfaces/package.py:1339
#: ../../lib/entropy/client/interfaces/package.py:3417
msgid "second"
msgstr "сек."

#: ../../lib/entropy/client/interfaces/package.py:684
#: ../../lib/entropy/client/interfaces/package.py:1355
#: ../../lib/entropy/client/interfaces/package.py:3432
msgid "Error downloading from"
msgstr "Ошибка при скачивании с"

#: ../../lib/entropy/client/interfaces/package.py:688
msgid "data not available on this mirror"
msgstr "информация недоступна на этом зеркале"

#: ../../lib/entropy/client/interfaces/package.py:691
#: ../../lib/entropy/client/interfaces/package.py:1364
msgid "wrong checksum"
msgstr "неверная контрольная сумма"

#. timeout!
#: ../../lib/entropy/client/interfaces/package.py:695
msgid "timeout error"
msgstr "ошибка при превышении времени ожидания"

#: ../../lib/entropy/client/interfaces/package.py:697
#: ../../lib/entropy/client/interfaces/package.py:1385
#: ../../lib/entropy/client/interfaces/package.py:3443
msgid "discarded download"
msgstr "отмененная загрузка"

#: ../../lib/entropy/client/interfaces/package.py:700
#: ../../lib/entropy/client/interfaces/package.py:1388
#: ../../lib/entropy/client/interfaces/package.py:3445
msgid "unknown reason"
msgstr "неизвестная причина"

#: ../../lib/entropy/client/interfaces/package.py:1311
msgid "Downloading from"
msgstr "Скачивание с"

#: ../../lib/entropy/client/interfaces/package.py:1333
#: ../../lib/entropy/client/interfaces/package.py:3414
msgid "Successfully downloaded from"
msgstr "Успешно скачано с"

#: ../../lib/entropy/client/interfaces/package.py:1338
#: ../../lib/entropy/client/interfaces/package.py:3417
msgid "at"
msgstr "на"

#: ../../lib/entropy/client/interfaces/package.py:1361
#: ../../lib/entropy/client/interfaces/package.py:3438
msgid "file not available on this mirror"
msgstr "файл недоступен на этом зеркале"

#: ../../lib/entropy/client/interfaces/package.py:1369
msgid "Disabling resume"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1379
msgid "timeout, retrying on this mirror"
msgstr "превышено время ожидания, повтор попытки на данном зеркале"

#: ../../lib/entropy/client/interfaces/package.py:1382
msgid "timeout, giving up"
msgstr "превышено время ожидания, прекращаю"

#: ../../lib/entropy/client/interfaces/package.py:1487
msgid "Package signature verification error for"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1519
msgid "disabled"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1532
msgid "Checking package signature"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1543
msgid "Package signature verification"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1545
msgid "temporarily unavailable"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1555
msgid "Package signature"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1557
msgid "does not match the recorded one"
msgstr "не подсчитывает записанный"

#: ../../lib/entropy/client/interfaces/package.py:1567
msgid "matches"
msgstr "подсчёты"

#: ../../lib/entropy/client/interfaces/package.py:1582
msgid "Checking package checksum..."
msgstr "Подсчёт контрольной суммы пакета..."

#: ../../lib/entropy/client/interfaces/package.py:1595
msgid "Package checksum matches"
msgstr "Контрольная сумма пакета корректна"

#: ../../lib/entropy/client/interfaces/package.py:1615
#, python-format
msgid "Checksum does not match. Download attempt #%s"
msgstr "Контрольная сумма пакета некорректна. Попытка загрузки #%s"

#: ../../lib/entropy/client/interfaces/package.py:1637
msgid "Cannot properly fetch package! Quitting."
msgstr "В общем, невозможно загрузить пакет! Выход."

#: ../../lib/entropy/client/interfaces/package.py:1649
msgid "Cannot fetch package or checksum does not match"
msgstr "Невозможно загрузить пакет либо контрольная сумма некорректна."

#: ../../lib/entropy/client/interfaces/package.py:1650
msgid "Try to download latest repositories"
msgstr "Попробуйте загрузить последние репозитории"

#: ../../lib/entropy/client/interfaces/package.py:1679
#: ../../lib/entropy/client/interfaces/package.py:4072
msgid "Unpacking"
msgstr "Распаковка"

#: ../../lib/entropy/client/interfaces/package.py:1720
msgid "Unpack error"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1749
msgid "Unable to find Entropy metadata in package"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1813
msgid "configuration phase"
msgstr "Фаза конфигурации"

#: ../../lib/entropy/client/interfaces/package.py:1830
msgid "Removing from Entropy"
msgstr "Removing from Entropy"

#: ../../lib/entropy/client/interfaces/package.py:1916
msgid "Removing config file, never modified"
msgstr "Удаление конфигурационного файла, никогда не модифицированного"

#: ../../lib/entropy/client/interfaces/package.py:1938
msgid "remove"
msgstr "удалить"

#: ../../lib/entropy/client/interfaces/package.py:1939
#: ../../lib/entropy/client/interfaces/package.py:3305
msgid "Protecting config file"
msgstr "Защита конфигурационного файла"

#: ../../lib/entropy/client/interfaces/package.py:1958
msgid "This package contains a badly encoded file !!!"
msgstr "Данный пакет содержит файл с испорченной кодировкой!!!"

#: ../../lib/entropy/client/interfaces/package.py:2075
msgid "Collision found during removal of"
msgstr "Произошла ошибка при удалении"

#: ../../lib/entropy/client/interfaces/package.py:2191
msgid "Updating database"
msgstr "Обновление базы данных"

#: ../../lib/entropy/client/interfaces/package.py:2685
#, python-format
msgid "%s is a file when should be a directory !! Removing in 20 seconds..."
msgstr ""
"%s - это файл, который должен быть папкой !! Удаляется в течение 20 секунд..."

#: ../../lib/entropy/client/interfaces/package.py:2710
msgid "directory expected, symlink found"
msgstr "найден ярлык, хотя нужна была папка"

#: ../../lib/entropy/client/interfaces/package.py:2713
#: ../../lib/entropy/client/interfaces/package.py:3017
msgid "Removing in 20 seconds !!"
msgstr "Удаление через 20 секунд !!"

#: ../../lib/entropy/client/interfaces/package.py:2759
msgid "Cannot remove symlink"
msgstr ""

#. we can merge it, files, even if
#. contains changes have not been modified
#. by the user
#: ../../lib/entropy/client/interfaces/package.py:2923
msgid "Automerging config file, never modified"
msgstr ""
"Автообъединение конфигурационного файла, никогда не модифицировавшегося"

#: ../../lib/entropy/client/interfaces/package.py:2952
#: ../../lib/entropy/client/interfaces/package.py:2974
msgid "Circular symlink issue"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3014
msgid "file expected, directory found"
msgstr "найдена папка, хотя нужен был файл"

#: ../../lib/entropy/client/interfaces/package.py:3067
msgid "File move error"
msgstr "Ошибка при перемещении файла"

#: ../../lib/entropy/client/interfaces/package.py:3070
msgid "please report"
msgstr "пожалуйста, отправьте отчет"

#: ../../lib/entropy/client/interfaces/package.py:3222
msgid "Cannot protect broken symlink"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3261
msgid "Skipping file installation/removal"
msgstr "Пропуск установки/удаления файла"

#: ../../lib/entropy/client/interfaces/package.py:3322
msgid "Collision found during install for"
msgstr "Обнаружено противоречие во время установки"

#: ../../lib/entropy/client/interfaces/package.py:3325
msgid "cannot overwrite"
msgstr "невозможно перезаписать"

#: ../../lib/entropy/client/interfaces/package.py:3398
#: ../../lib/entropy/client/interfaces/package.py:3461
#: ../../lib/entropy/client/interfaces/package.py:3508
#: ../../client/solo/utils.py:253 ../../client/solo/commands/command.py:286
msgid "Downloading"
msgstr "Скачивание"

#: ../../lib/entropy/client/interfaces/package.py:3425
msgid "Local path"
msgstr "Локальный путь"

#: ../../lib/entropy/client/interfaces/package.py:3491
msgid "Package cannot be fetched. Try to update repositories"
msgstr "Пакет не может быть загружен. Попробуйте обновить репозитории."

#: ../../lib/entropy/client/interfaces/package.py:3510
msgid "archive"
msgid_plural "archives"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/package.py:3527
msgid "Some packages cannot be fetched"
msgstr "Некоторые пакеты не могут быть загружены"

#: ../../lib/entropy/client/interfaces/package.py:3528
msgid "Try to update your repositories and retry"
msgstr "Попробуйте обновить репозитории и попробовать снова"

#: ../../lib/entropy/client/interfaces/package.py:3556
msgid "Package cannot be downloaded, unknown error."
msgstr "Пакет не может быть скачан, неизвестная ошибка."

#: ../../lib/entropy/client/interfaces/package.py:3565
msgid "Installed package in queue vanished, skipping."
msgstr "Установленные пакеты в очереди очищены, пропускаем."

#: ../../lib/entropy/client/interfaces/package.py:3601
msgid "Merging package"
msgstr "Соединение пакета"

#: ../../lib/entropy/client/interfaces/package.py:3654
msgid "You are running out of disk space"
msgstr "Осталось совсем мало места на диске"

#: ../../lib/entropy/client/interfaces/package.py:3655
msgid "I bet, you're probably Michele"
msgstr "Спорим, что ты Микеле"

#: ../../lib/entropy/client/interfaces/package.py:3659
msgid "An error occured while trying to unpack the package"
msgstr "Произошла ошибка при попытке распаковать пакет"

#: ../../lib/entropy/client/interfaces/package.py:3662
#: ../../lib/entropy/client/interfaces/package.py:3718
msgid "Check if your system is healthy"
msgstr "Проверьте, всё ли в порядке с системой"

#: ../../lib/entropy/client/interfaces/package.py:3681
msgid "Installing package"
msgstr "Установка пакета"

#: ../../lib/entropy/client/interfaces/package.py:3701
msgid "unsupported splitdebug usage (package files)"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3705
msgid "<3 debug files installation enabled <3"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3717
msgid "An error occured while trying to install the package"
msgstr "Произошла ошибка при попытке установить пакет"

#: ../../lib/entropy/client/interfaces/package.py:3732
msgid "Removing data"
msgstr "Удаление информации"

#: ../../lib/entropy/client/interfaces/package.py:3745
msgid "An error occured while trying to remove the package"
msgstr "Произошла ошибка при попытке удалить пакет"

#: ../../lib/entropy/client/interfaces/package.py:3746
msgid "Check if you have enough disk space on your hard disk"
msgstr "Проверьте, достаточно ли места на жёстком диске"

#: ../../lib/entropy/client/interfaces/package.py:3760
#: ../../lib/entropy/client/interfaces/package.py:4108
#: ../../client/solo/utils.py:64
msgid "Cleaning"
msgstr "Очистка"

#: ../../lib/entropy/client/interfaces/package.py:3819
msgid "Error during package files permissions setup"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3851
msgid "Cleaning previously installed application data."
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3958
msgid "Configuring package"
msgstr "Конфигурирование пакета"

#: ../../lib/entropy/client/interfaces/package.py:3970
#: ../../lib/entropy/client/interfaces/package.py:3990
msgid "An error occured while trying to configure the package"
msgstr "Случилась ошибка при попытке конфигурирования пакета"

#: ../../lib/entropy/client/interfaces/package.py:3972
msgid "Make sure that your system is healthy"
msgstr "Будьте уверены, что с Вашей системой всё в порядке"

#: ../../lib/entropy/client/interfaces/package.py:3992
msgid "It seems that Source Package Manager entry is missing"
msgstr "Это означает, что недостаёт записи Менеджера Пакетов Источника"

#: ../../lib/entropy/client/interfaces/package.py:4016
msgid "Installed package vanished"
msgstr "Установленный пакет исчез"

#: ../../lib/entropy/client/interfaces/package.py:4022
msgid "Fetch not available"
msgstr "Загрузка недоступна"

#: ../../lib/entropy/client/interfaces/package.py:4029
msgid "Fetching"
msgstr "Загрузка"

#: ../../lib/entropy/client/interfaces/package.py:4037
msgid "Multi Fetching"
msgstr "Мульти-загрузка"

#: ../../lib/entropy/client/interfaces/package.py:4038
#: ../../lib/entropy/client/interfaces/package.py:4058
#: ../../client/solo/commands/query.py:1265
#: ../../client/solo/commands/_manage.py:602
msgid "package"
msgid_plural "packages"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/package.py:4044
msgid "Fetching sources"
msgstr "Загрузка источников"

#: ../../lib/entropy/client/interfaces/package.py:4050
msgid "Verifying"
msgstr "Проверка"

#: ../../lib/entropy/client/interfaces/package.py:4057
msgid "Multi Verification"
msgstr "Мульти-проверка"

#: ../../lib/entropy/client/interfaces/package.py:4064
msgid "Merging"
msgstr "Объединение"

#: ../../lib/entropy/client/interfaces/package.py:4085
msgid "Installing"
msgstr "Установка"

#: ../../lib/entropy/client/interfaces/package.py:4116
msgid "Postinstall"
msgstr "Завершение установки"

#: ../../lib/entropy/client/interfaces/package.py:4124
msgid "Setup"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:4132
msgid "Preinstall"
msgstr "Подготовка к установке"

#: ../../lib/entropy/client/interfaces/package.py:4140
msgid "Preremove"
msgstr "Подготовка к удалению"

#: ../../lib/entropy/client/interfaces/package.py:4148
msgid "Postremove"
msgstr "Завершение удаления"

#: ../../lib/entropy/client/interfaces/package.py:4162
msgid "Configuring"
msgstr "Конфигурирование"

#: ../../lib/entropy/client/interfaces/package.py:4215
msgid "An error occured. Action aborted."
msgstr "Произошла ошибка. Действие отменено."

#: ../../lib/entropy/client/interfaces/client.py:950
msgid "forcing packages metadata update"
msgstr "принуждение к обновлению метаинформации пакетов"

#: ../../lib/entropy/client/interfaces/client.py:959
msgid "Updating system database using repository"
msgstr "Обновление системной базы данных, используя ID репозитория"

#: ../../lib/entropy/client/interfaces/trigger.py:73
msgid "Source Package Manager interface can't be loaded"
msgstr "Пакетный менеджер источника не может быть загружен"

#: ../../lib/entropy/client/interfaces/trigger.py:76
msgid "please fix"
msgstr "пожалуйста, пофиксите"

#: ../../lib/entropy/client/interfaces/trigger.py:281
msgid "Cannot run External trigger for"
msgstr "Невозможно запустить Внешний триггер для"

#: ../../lib/entropy/client/interfaces/trigger.py:503
msgid "Installing info"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:522
msgid "post-install phase"
msgstr "фаза постустановки"

#: ../../lib/entropy/client/interfaces/trigger.py:532
msgid "pre-install phase"
msgstr "фаза предустановки"

#: ../../lib/entropy/client/interfaces/trigger.py:542
msgid "setup phase"
msgstr "фаза установки"

#: ../../lib/entropy/client/interfaces/trigger.py:552
msgid "pre-remove phase"
msgstr "фаза предудаления"

#: ../../lib/entropy/client/interfaces/trigger.py:562
msgid "post-remove phase"
msgstr "фаза постудаления"

#: ../../lib/entropy/client/interfaces/repository.py:172
msgid "Something bad happened. Please have a look."
msgstr "Случилось что-то нехорошее. Пожалуйста, взгляните."

#: ../../lib/entropy/client/interfaces/repository.py:194
msgid "there is"
msgid_plural "there are"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/repository.py:196
msgid "package file that could be removed"
msgid_plural "package files that could be removed"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/repository.py:209
msgid "Packages are stored in"
msgstr ""

#: ../../lib/entropy/client/interfaces/repository.py:239
msgid "a new release is available"
msgstr "доступна новая версия"

#: ../../lib/entropy/client/interfaces/repository.py:240
msgid "Mind to install it before any other package"
msgstr "Не забудьте установить это перед любыми другими пакетами"

#: ../../lib/entropy/client/interfaces/repository.py:305
msgid "Advisories fetch error"
msgstr "Ошибка при загрузке уведомлений"

#. let's dance!
#: ../../lib/entropy/client/interfaces/repository.py:337
msgid "Repositories synchronization"
msgstr "Синхронизация репозитория"

#: ../../lib/entropy/client/interfaces/dep.py:51
#, python-format
msgid "Checking %s"
msgstr "Проверка %s"

#: ../../lib/entropy/client/interfaces/dep.py:1803
msgid "Sorting dependencies"
msgstr "Сортировка зависимостей"

#: ../../lib/entropy/client/interfaces/dep.py:2004
msgid "Calculating inverse dependencies for"
msgstr "Подсчет противоположных зависимостей для"

#: ../../lib/entropy/client/interfaces/dep.py:2617
msgid "Calculating updates"
msgstr ""

#: ../../lib/entropy/fetchers.py:792
msgid "infinite"
msgstr "бесконечно"

#: ../../lib/entropy/fetchers.py:864
msgid "[F]"
msgstr "[F]"

#: ../../lib/entropy/fetchers.py:865 ../../lib/entropy/fetchers.py:1239
msgid "ETA"
msgstr "ЕТА"

#: ../../lib/entropy/fetchers.py:866 ../../lib/entropy/fetchers.py:1240
msgid "sec"
msgstr "сек"

#: ../../lib/entropy/fetchers.py:1121
msgid "Aggregated download"
msgstr "Аггрегированная загрузка"

#: ../../lib/entropy/fetchers.py:1123
msgid "item"
msgid_plural "items"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/security.py:355
msgid "GPG key EXPIRED for URL"
msgstr ""

#: ../../lib/entropy/security.py:366
msgid "Installing GPG key for URL"
msgstr ""

#: ../../lib/entropy/security.py:391
msgid "Successfully installed GPG key for URL"
msgstr ""

#: ../../lib/entropy/security.py:445 ../../lib/entropy/security.py:1020
#: ../../lib/entropy/security.py:1035 ../../lib/entropy/security.py:1042
#: ../../lib/entropy/security.py:1084 ../../lib/entropy/security.py:1107
#: ../../lib/entropy/security.py:1119 ../../lib/entropy/security.py:1136
#: ../../lib/entropy/security.py:1148 ../../lib/entropy/security.py:1160
#: ../../lib/entropy/security.py:1172 ../../lib/entropy/security.py:1193
#: ../../lib/entropy/security.py:1221 ../../lib/entropy/security.py:1233
msgid "Security Advisories"
msgstr "Предостережения по безопасности"

#: ../../lib/entropy/security.py:446
msgid "GPG key verification successful"
msgstr ""

#: ../../lib/entropy/security.py:662 ../../client/solo/commands/pkg.py:195
#: ../../client/solo/commands/_manage.py:457
#: ../../client/solo/commands/_manage.py:479
#: ../../client/solo/commands/mask.py:99
#: ../../client/solo/commands/mask.py:172
msgid "Warning"
msgstr "Осторожно"

#: ../../lib/entropy/security.py:664
msgid "advisory broken"
msgstr "уведомление испорчено"

#: ../../lib/entropy/security.py:1021
msgid "getting latest advisories"
msgstr ""

#: ../../lib/entropy/security.py:1036
msgid "updated successfully"
msgstr "успешно обновлено"

#: ../../lib/entropy/security.py:1043
msgid "already up to date"
msgstr "уже обновлено до последней версии"

#: ../../lib/entropy/security.py:1085
msgid "cannot download checksum, sorry"
msgstr ""

#: ../../lib/entropy/security.py:1108
msgid "unable to download advisories, sorry"
msgstr ""

#: ../../lib/entropy/security.py:1120
msgid "Verifying checksum"
msgstr "Проверка контрольной суммы"

#: ../../lib/entropy/security.py:1137
msgid "cannot open packages, sorry"
msgstr "увы, невозможно открыть пакеты"

#: ../../lib/entropy/security.py:1149
msgid "cannot read the checksum, sorry"
msgstr "жаль, но невозможно прочитать контрольную сумму"

#: ../../lib/entropy/security.py:1161
msgid "digest verification failed, sorry"
msgstr "Не удалось проверить списки, увы"

#: ../../lib/entropy/security.py:1173
msgid "verification successful"
msgstr ""

#: ../../lib/entropy/security.py:1194
msgid "GPG service not available"
msgstr ""

#: ../../lib/entropy/security.py:1222
msgid "digest verification failed, try again later"
msgstr "проверка списков не удалась, попробуйте позже"

#: ../../lib/entropy/security.py:1234
msgid "installing"
msgstr "установка"

#: ../../lib/entropy/const.py:661
msgid "Runtime dependency"
msgstr ""

#: ../../lib/entropy/const.py:662
msgid "Post dependency"
msgstr ""

#: ../../lib/entropy/const.py:663
msgid "Manually added (by staff) dependency"
msgstr ""

#: ../../lib/entropy/const.py:664
msgid "Build dependency"
msgstr ""

#: ../../lib/entropy/const.py:684
msgid "Corrupted Client Repository. Please restore a backup."
msgstr ""
"Испорчен клиент репозитория. Пожалуйста, восстановите из резервной копии."

#: ../../lib/entropy/const.py:716
msgid "reason not available"
msgstr "причина недоступна"

#: ../../lib/entropy/const.py:717
msgid "user package.mask"
msgstr "package.mask пользователя"

#: ../../lib/entropy/const.py:718
msgid "system keywords"
msgstr "системные ключевые слова"

#: ../../lib/entropy/const.py:719
msgid "user package.unmask"
msgstr "package.unmask пользователя"

#: ../../lib/entropy/const.py:720
msgid "user repo package.keywords (all packages)"
msgstr "user repo package.keywords (все пакеты)"

#: ../../lib/entropy/const.py:721
msgid "user repo package.keywords"
msgstr "user repo package.keywords"

#: ../../lib/entropy/const.py:722
msgid "user package.keywords"
msgstr "package.keywords пользователя"

#: ../../lib/entropy/const.py:723
msgid "completely masked (by keyword?)"
msgstr "полностью замаскировано (по ключевому слову?)"

#: ../../lib/entropy/const.py:724
msgid "repository general packages.db.mask"
msgstr "repository general packages.db.mask"

#: ../../lib/entropy/const.py:725
msgid "repository general packages.db.keywords"
msgstr "repository in branch packages.db.mask"

#: ../../lib/entropy/const.py:726
msgid "user license.mask"
msgstr "user license.mask"

#: ../../lib/entropy/const.py:727
msgid "user live unmask"
msgstr "user live unmask"

#: ../../lib/entropy/const.py:728
msgid "user live mask"
msgstr "user live mask"

#: ../../lib/entropy/output.py:643
msgid "Entropy needs your attention"
msgstr "Entropy требует Вашего внимания"

#: ../../lib/entropy/output.py:781 ../../lib/entropy/output.py:1026
#: ../../client/solo/commands/ugc.py:650 ../../client/solo/commands/ugc.py:760
#: ../../client/solo/commands/ugc.py:783 ../../client/solo/commands/ugc.py:964
#: ../../client/solo/commands/upgrade.py:249
#: ../../client/solo/commands/upgrade.py:277
#: ../../client/solo/commands/rescue.py:654
#: ../../client/solo/commands/rescue.py:659
#: ../../client/solo/commands/rescue.py:699
msgid "Yes"
msgstr "Да"

#: ../../lib/entropy/output.py:781 ../../client/solo/main.py:169
#: ../../client/solo/commands/pkg.py:253
#: ../../client/solo/commands/config.py:145
#: ../../client/solo/commands/remove.py:381
#: ../../client/solo/commands/remove.py:388
#: ../../client/solo/commands/remove.py:433
#: ../../client/solo/commands/deptest.py:180
#: ../../client/solo/commands/libtest.py:220
#: ../../client/solo/commands/_manage.py:242
#: ../../client/solo/commands/conf.py:391
#: ../../client/solo/commands/conf.py:446
#: ../../client/solo/commands/install.py:517
#: ../../client/solo/commands/upgrade.py:249
#: ../../client/solo/commands/rescue.py:326
#: ../../client/solo/commands/rescue.py:330
#: ../../client/solo/commands/rescue.py:337
#: ../../client/solo/commands/mask.py:161
msgid "No"
msgstr "Нет"

#: ../../lib/entropy/output.py:796
msgid "Entropy got a question for you"
msgstr "у Entropy есть к Вам вопросы"

#: ../../lib/entropy/output.py:810
msgid "Interrupted"
msgstr "Отменено"

#: ../../lib/entropy/output.py:872 ../../lib/entropy/output.py:876
msgid "Selected number"
msgstr "Выбранный номер"

#: ../../lib/entropy/output.py:893 ../../client/solo/commands/_manage.py:714
msgid "Please select an option"
msgstr "Пожалуйста, выберите опцию"

#: ../../lib/entropy/output.py:895
msgid "Discard all"
msgstr "Отменить всё"

#: ../../lib/entropy/output.py:896
msgid "Confirm"
msgstr "Подтвердить"

#: ../../lib/entropy/output.py:897
msgid "Add item"
msgstr "Добавить предмет"

#: ../../lib/entropy/output.py:898
msgid "Edit item"
msgstr ""

#: ../../lib/entropy/output.py:899
msgid "Remove item"
msgstr "Удалить предмет"

#: ../../lib/entropy/output.py:900
msgid "Show current list"
msgstr "Показать текущий список"

#: ../../lib/entropy/output.py:904 ../../client/solo/commands/conf.py:247
#: ../../client/solo/commands/conf.py:290
msgid "Your choice (type a number and press enter):"
msgstr "Ваш выбор (введите номер и нажмите Enter)"

#: ../../lib/entropy/output.py:936
msgid "You don't have typed a number."
msgstr "Вы не ввели номер."

#: ../../lib/entropy/output.py:939
msgid "Invalid action."
msgstr "Неправильное действие."

#: ../../lib/entropy/output.py:949
msgid "String to add (-1 to go back):"
msgstr ""

#: ../../lib/entropy/output.py:959
msgid "Invalid string."
msgstr "Неправильная строка."

#: ../../lib/entropy/output.py:967
msgid "Element number to edit (-1 to go back):"
msgstr ""

#: ../../lib/entropy/output.py:978
msgid "old"
msgstr ""

#: ../../lib/entropy/output.py:978
msgid "new value:"
msgstr ""

#: ../../lib/entropy/output.py:986 ../../lib/entropy/output.py:1004
msgid "Invalid element."
msgstr "Неправильный элемент."

#: ../../lib/entropy/output.py:995
msgid "Element number to remove (-1 to go back):"
msgstr ""

#: ../../lib/entropy/qa.py:161
msgid "Searching for broken reverse dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:178
msgid "scanning for broken reverse dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:219
msgid "broken libraries detected"
msgstr "испорченные библиотеки найдены"

#: ../../lib/entropy/qa.py:230
msgid "needs"
msgstr "требует"

#: ../../lib/entropy/qa.py:274
msgid "Searching for missing Runtime dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:281
msgid "scanning"
msgstr ""

#: ../../lib/entropy/qa.py:299 ../../lib/entropy/qa.py:587
msgid "no packages"
msgstr ""

#: ../../lib/entropy/qa.py:351
msgid "blacklisted dependencies !!!"
msgstr ""

#: ../../lib/entropy/qa.py:389
msgid "is potentially missing these dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:526
msgid "determining missing libraries"
msgstr ""

#: ../../lib/entropy/qa.py:558
msgid "requires"
msgstr ""

#: ../../lib/entropy/qa.py:575 ../../lib/entropy/qa.py:586
msgid "provided by"
msgstr ""

#: ../../lib/entropy/qa.py:599
msgid "no missing runtime libraries found"
msgstr ""

#: ../../lib/entropy/qa.py:635
msgid "Libraries test"
msgstr "Тест библиотек"

#: ../../lib/entropy/qa.py:650
msgid "Broken symbols packages list"
msgstr "Список испорченных пакетов символов"

#: ../../lib/entropy/qa.py:651
msgid "Broken executables list"
msgstr "Список испорченных исполняемых файлов"

#: ../../lib/entropy/qa.py:653
msgid "Dumping results into these files"
msgstr "Запись результатов в эти файлы"

#: ../../lib/entropy/qa.py:733
msgid "discarding directory"
msgstr "папка забракованного"

#: ../../lib/entropy/qa.py:735
msgid "because it's symlinked on"
msgstr "поскольку оно ссылается на"

#: ../../lib/entropy/qa.py:797
msgid "Collecting broken executables"
msgstr "Ищутся испорченные исполняемые файлы"

#: ../../lib/entropy/qa.py:803
msgid "don't worry about libraries that are shown here but not later."
msgstr "не беспокойтесь о библиотеках, которые показаны здесь, но не позже"

#: ../../lib/entropy/qa.py:824
msgid "Scanning libraries"
msgstr "Сканирование библиотек"

#: ../../lib/entropy/qa.py:932
msgid "various broken symbols"
msgstr "Различные испорченные символы"

#: ../../lib/entropy/qa.py:973
msgid "Matching broken libraries/executables"
msgstr "Подсчет испорченных библиотек/исполняемых приложений"

#: ../../lib/entropy/qa.py:1677
msgid "Not prepared yet"
msgstr "Ещё не подготовлено."

#: ../../client/solo/utils.py:65
msgid "directory"
msgstr ""

#: ../../client/solo/utils.py:73
msgid "Cleaned"
msgstr ""

#: ../../client/solo/utils.py:75
msgid "files and directories"
msgstr ""

#: ../../client/solo/utils.py:176 ../../client/solo/utils.py:319
#: ../../client/solo/utils.py:320 ../../client/solo/commands/repo.py:466
msgid "N/A"
msgstr "N/A"

#: ../../client/solo/utils.py:255 ../../client/solo/commands/command.py:288
msgid "Uploading"
msgstr ""

#: ../../client/solo/utils.py:277 ../../client/solo/commands/rescue.py:233
msgid "corrupted"
msgstr ""

#: ../../client/solo/utils.py:318 ../../client/solo/commands/_manage.py:150
#: ../../client/solo/commands/_manage.py:194
msgid "Not installed"
msgstr "Не установлено"

#: ../../client/solo/utils.py:344 ../../client/solo/commands/query.py:453
#: ../../client/solo/commands/query.py:696
#: ../../client/solo/commands/query.py:1021
msgid "Package"
msgstr "Пакет"

#: ../../client/solo/utils.py:346 ../../client/solo/commands/hop.py:57
msgid "branch"
msgstr "ветка"

#: ../../client/solo/utils.py:353
msgid "Category"
msgstr "Категория"

#: ../../client/solo/utils.py:355
msgid "Name"
msgstr "Имя"

#: ../../client/solo/utils.py:371 ../../client/solo/commands/query.py:561
msgid "Masked"
msgstr "Замаскировано"

#: ../../client/solo/utils.py:374
msgid "Available"
msgstr "Доступно"

#: ../../client/solo/utils.py:376 ../../client/solo/utils.py:384
msgid "Installed"
msgstr "Установлено"

#: ../../client/solo/utils.py:379 ../../client/solo/utils.py:385
msgid "version"
msgstr "версия"

#: ../../client/solo/utils.py:380 ../../client/solo/utils.py:387
msgid "revision"
msgstr "версия"

#: ../../client/solo/utils.py:390
msgid "Slot"
msgstr "Слот"

#: ../../client/solo/utils.py:406
msgid "Size"
msgstr "Размер"

#: ../../client/solo/utils.py:408 ../../client/solo/commands/ugc.py:376
msgid "Download"
msgstr "Загрузка"

#: ../../client/solo/utils.py:410
msgid "Checksum"
msgstr "Контрольная сумма"

#: ../../client/solo/utils.py:415
msgid "Dependencies"
msgstr "Зависимости"

#: ../../client/solo/utils.py:424 ../../client/solo/utils.py:607
msgid "Legend"
msgstr ""

#: ../../client/solo/utils.py:432
msgid "Conflicts"
msgstr "Конфликты"

#: ../../client/solo/utils.py:437
msgid "Homepage"
msgstr "Домашняя страница"

#: ../../client/solo/utils.py:447 ../../client/solo/commands/security.py:220
#: ../../client/solo/commands/ugc.py:621
msgid "Description"
msgstr "Описание"

#: ../../client/solo/utils.py:456
msgid "USE flags"
msgstr "USE-флаги"

#: ../../client/solo/utils.py:476
msgid "CHOST"
msgstr "CHOST"

#: ../../client/solo/utils.py:478
msgid "CFLAGS"
msgstr "CFLAGS"

#: ../../client/solo/utils.py:480
msgid "CXXFLAGS"
msgstr "CXXFLAGS"

#: ../../client/solo/utils.py:484
msgid "Sources"
msgstr "Источники"

#: ../../client/solo/utils.py:489
msgid "Entry API"
msgstr "Запись API"

#: ../../client/solo/utils.py:491
msgid "Compiled with"
msgstr "Компилировано с"

#: ../../client/solo/utils.py:496 ../../client/solo/commands/search.py:178
#: ../../client/solo/commands/ugc.py:369 ../../client/solo/commands/ugc.py:628
#: ../../client/solo/commands/match.py:203
msgid "Keywords"
msgstr "Ключевые слова"

#: ../../client/solo/utils.py:511
msgid "Created"
msgstr "Создано"

#: ../../client/solo/utils.py:517 ../../client/solo/commands/_manage.py:773
msgid "License"
msgstr "Лицензия"

#: ../../client/solo/utils.py:539
msgid "Reverse graphing installed package"
msgstr ""

#: ../../client/solo/utils.py:551 ../../client/solo/utils.py:710
#: ../../client/solo/commands/config.py:112
#: ../../client/solo/commands/remove.py:362
#: ../../client/solo/commands/source.py:134
#: ../../client/solo/commands/install.py:492
#: ../../client/solo/commands/download.py:136
#: ../../client/solo/commands/mask.py:118
msgid "No packages found"
msgstr "Пакетов не найдено"

#: ../../client/solo/utils.py:610
msgid "packages passed as arguments"
msgstr ""

#: ../../client/solo/utils.py:613
msgid "packages with no further dependencies"
msgstr ""

#: ../../client/solo/utils.py:616
msgid "packages with further dependencies (node)"
msgstr ""

#: ../../client/solo/utils.py:619
msgid "packages already pulled in as dependency in upper levels (circularity)"
msgstr ""

#: ../../client/solo/utils.py:698
msgid "Graphing"
msgstr ""

#: ../../client/solo/main.py:41
msgid "Installed packages repository corrupted. Please re-generate it"
msgstr ""

#: ../../client/solo/main.py:51
msgid "Cannot continue"
msgstr "Невозможно продолжить"

#: ../../client/solo/main.py:80
msgid "Your hard drive is full! Your fault!"
msgstr ""

#: ../../client/solo/main.py:89
msgid "No more memory dude! Your fault!"
msgstr ""

#: ../../client/solo/main.py:95
msgid ""
"Hi. My name is Bug Reporter. I am sorry to inform you that the program "
"crashed. Well, you know, shit happens."
msgstr ""

#: ../../client/solo/main.py:101
msgid ""
"But there's something you could do to help me to be a better application."
msgstr ""

#: ../../client/solo/main.py:107
msgid "-- BUT, DO NOT SUBMIT THE SAME REPORT MORE THAN ONCE --"
msgstr ""

#: ../../client/solo/main.py:112
msgid "Now I am showing you what happened. Don't panic, I'm here to help you."
msgstr ""
"Теперь я покажу тебе, что случилось. Не паникуй, я здесь, чтобы помочь тебе. "

#: ../../client/solo/main.py:149
msgid ""
"Oh well, I cannot even write to TMPDIR. So, please copy the error and mail "
"lxnay@sabayon.org."
msgstr ""

#: ../../client/solo/main.py:164
msgid ""
"Erm... Can I send the error, along with some other information\n"
"about your hardware to my creators so they can fix me? (Your IP will be "
"logged)"
msgstr ""

#: ../../client/solo/main.py:171
msgid "Ok, ok ok ok... Sorry!"
msgstr "Ну ладно, ладно, ладно, хорошо, ХОРОШО, ИЗВИНИ!"

#: ../../client/solo/main.py:177
msgid ""
"If you want to be contacted back (and actively supported), also answer the "
"questions below:"
msgstr ""
"Когда будешь связываться (и активно общаться с техподдержкой), ответь заодно "
"на нижеуказанные вопросы:"

#: ../../client/solo/main.py:184
msgid "Your Full name:"
msgstr "Ваше полное имя:"

#: ../../client/solo/main.py:185
msgid "Your E-Mail address:"
msgstr "Ваш адрес E-Mail:"

#: ../../client/solo/main.py:186
msgid "What you were doing:"
msgstr "Чего ж ты такого наделал:"

#: ../../client/solo/main.py:208
msgid ""
"Thank you very much. The error has been reported and hopefully, the problem "
"will be solved as soon as possible."
msgstr ""
"Спасибо тебе огромное. Данные об ошибке отправлены и, если всё будет "
"нормально, проблема будет решена как можно раньше."

#: ../../client/solo/main.py:214
msgid ""
"Ugh. Cannot send the report. Please mail the file below to lxnay@sabayon.org."
msgstr ""

#: ../../client/solo/main.py:234
msgid "Entropy/Equo version mismatch"
msgstr ""

#: ../../client/solo/main.py:235
msgid "it could make your system explode!"
msgstr ""

#: ../../client/solo/main.py:244
msgid "Entropy is running off a Live System"
msgstr ""

#: ../../client/solo/main.py:246
msgid "Performance and stability could get severely compromised"
msgstr ""

#: ../../client/solo/main.py:329
msgid "superuser access required"
msgstr ""

#: ../../client/solo/commands/pkg.py:72 ../../client/solo/commands/pkg.py:477
#: ../../client/solo/commands/rescue.py:71
msgid "execute advanced tasks on packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:73 ../../client/solo/commands/query.py:73
#: ../../client/solo/commands/query.py:154
#: ../../client/solo/commands/security.py:69
#: ../../client/solo/commands/cache.py:61 ../../client/solo/commands/ugc.py:72
#: ../../client/solo/commands/ugc.py:105 ../../client/solo/commands/ugc.py:144
#: ../../client/solo/commands/repo.py:66 ../../client/solo/commands/conf.py:76
#: ../../client/solo/commands/rescue.py:72 ../../services/kernel-switcher:317
msgid "available commands"
msgstr ""

#: ../../client/solo/commands/pkg.py:79
#: ../../client/solo/commands/config.py:63
#: ../../client/solo/commands/security.py:90
#: ../../client/solo/commands/security.py:160
#: ../../client/solo/commands/remove.py:67
#: ../../client/solo/commands/deptest.py:61
#: ../../client/solo/commands/source.py:62
#: ../../client/solo/commands/libtest.py:61
#: ../../client/solo/commands/install.py:70
#: ../../client/solo/commands/upgrade.py:65
#: ../../client/solo/commands/rescue.py:78
#: ../../client/solo/commands/download.py:62
#: ../../client/solo/commands/mask.py:49
msgid "ask before making any changes"
msgstr "спрашивать перед любыми изменениями"

#: ../../client/solo/commands/pkg.py:85
#: ../../client/solo/commands/command.py:81
msgid "not a valid Entropy package file"
msgstr ""

#: ../../client/solo/commands/pkg.py:90
msgid "generate packages from the installed system"
msgstr ""

#: ../../client/solo/commands/pkg.py:94
msgid "installed package name"
msgstr ""

#: ../../client/solo/commands/pkg.py:99 ../../client/solo/commands/pkg.py:116
#: ../../client/solo/commands/pkg.py:133 ../../client/solo/commands/pkg.py:150
msgid "destination directory where to save generated packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:106
msgid "transform SPM package files into Entropy ones"
msgstr ""

#: ../../client/solo/commands/pkg.py:111
msgid "SPM package file path"
msgstr ""

#: ../../client/solo/commands/pkg.py:123
msgid "transform Entropy package files into SPM ones"
msgstr ""

#: ../../client/solo/commands/pkg.py:128 ../../client/solo/commands/pkg.py:145
msgid "Entropy package file path"
msgstr ""

#: ../../client/solo/commands/pkg.py:140
msgid "extract Entropy metadata from Entropy packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:223 ../../client/solo/commands/pkg.py:301
#: ../../client/solo/commands/pkg.py:392 ../../client/solo/commands/pkg.py:439
msgid "broken directory path"
msgstr ""

#: ../../client/solo/commands/pkg.py:236
msgid "This is the list of packages that would be considered"
msgstr ""

#: ../../client/solo/commands/pkg.py:252
#: ../../client/solo/commands/config.py:144
#: ../../client/solo/commands/_manage.py:241
#: ../../client/solo/commands/install.py:516
msgid "Would you like to continue ?"
msgstr "Желаете ли продолжить?"

#: ../../client/solo/commands/pkg.py:263
msgid "generating package"
msgstr ""

#: ../../client/solo/commands/pkg.py:273
msgid "package file creation error"
msgstr ""

#: ../../client/solo/commands/pkg.py:281
msgid "package generated"
msgstr ""

#: ../../client/solo/commands/pkg.py:310 ../../client/solo/commands/pkg.py:399
#: ../../client/solo/commands/pkg.py:446
msgid "working on package file"
msgstr ""

#: ../../client/solo/commands/pkg.py:323
msgid "package file extraction complete"
msgstr ""

#: ../../client/solo/commands/pkg.py:361
msgid "package metadata generation complete"
msgstr ""

#: ../../client/solo/commands/pkg.py:372
msgid "package file generated at"
msgstr ""

#: ../../client/solo/commands/pkg.py:411 ../../client/solo/commands/pkg.py:459
msgid "error during metadata extraction"
msgstr ""

#: ../../client/solo/commands/pkg.py:419
msgid "package file generated"
msgstr ""

#: ../../client/solo/commands/pkg.py:467
msgid "metadata file generated"
msgstr ""

#: ../../client/solo/commands/query.py:72
#: ../../client/solo/commands/query.py:1538
msgid "repository query tools"
msgstr ""

#: ../../client/solo/commands/query.py:77
msgid "resolve what package a file belongs to"
msgstr ""

#: ../../client/solo/commands/query.py:80
#: ../../client/solo/commands/query.py:200
msgid "file path"
msgstr ""

#: ../../client/solo/commands/query.py:87
msgid "show package changelog"
msgstr ""

#: ../../client/solo/commands/query.py:90
#: ../../client/solo/commands/query.py:100
#: ../../client/solo/commands/query.py:120
#: ../../client/solo/commands/query.py:130
#: ../../client/solo/commands/query.py:215
#: ../../client/solo/commands/query.py:282
#: ../../client/solo/commands/query.py:299
#: ../../client/solo/commands/config.py:57
#: ../../client/solo/commands/remove.py:61
#: ../../client/solo/commands/source.py:56
#: ../../client/solo/commands/install.py:64
#: ../../client/solo/commands/download.py:56
#: ../../client/solo/commands/mask.py:44
msgid "package name"
msgstr ""

#: ../../client/solo/commands/query.py:97
msgid "show reverse dependencies of package"
msgstr ""

#: ../../client/solo/commands/query.py:107
msgid "search package by description"
msgstr ""

#: ../../client/solo/commands/query.py:110
msgid "description keyword"
msgstr ""

#: ../../client/solo/commands/query.py:117
msgid "show files owned by package"
msgstr ""

#: ../../client/solo/commands/query.py:127
msgid "search installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:137
msgid "show packages using the given license"
msgstr ""

#: ../../client/solo/commands/query.py:140
msgid "license name"
msgstr ""

#: ../../client/solo/commands/query.py:146
#: ../../client/solo/commands/query.py:153
msgid "list packages"
msgstr ""

#: ../../client/solo/commands/query.py:157
msgid "list installed packages"
msgstr "показать установленные пакеты"

#: ../../client/solo/commands/query.py:160
msgid "only list packages installed by user"
msgstr ""

#: ../../client/solo/commands/query.py:163
msgid "only list packages installed from given repositories"
msgstr ""

#: ../../client/solo/commands/query.py:171
msgid "list available packages"
msgstr "показать доступные пакеты"

#: ../../client/solo/commands/query.py:174
msgid "only list packages from given repositories"
msgstr ""

#: ../../client/solo/commands/query.py:182
msgid "show packages able to handle the given mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:185
msgid "mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:188
#: ../../client/solo/commands/query.py:203
msgid "only show installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:197
msgid "associate file to packages able to handle it"
msgstr ""

#: ../../client/solo/commands/query.py:212
msgid "show runtime libraries needed by the given package"
msgstr ""

#: ../../client/solo/commands/query.py:222
msgid "search files not belonging to any packages"
msgstr ""

#: ../../client/solo/commands/query.py:228
msgid "show packages requiring the given library name"
msgstr ""

#: ../../client/solo/commands/query.py:231
msgid "library name (example: libdl.so.2)"
msgstr ""

#: ../../client/solo/commands/query.py:241
msgid "set name"
msgstr ""

#: ../../client/solo/commands/query.py:248
msgid "show packages owning the given slot"
msgstr ""

#: ../../client/solo/commands/query.py:251
msgid "slot name"
msgstr ""

#: ../../client/solo/commands/query.py:258
msgid "show packages owning the given tag"
msgstr ""

#: ../../client/solo/commands/query.py:261
msgid "tag name"
msgstr ""

#: ../../client/solo/commands/query.py:268
msgid "show packages at the given revision"
msgstr ""

#: ../../client/solo/commands/query.py:271
msgid "revision name"
msgstr ""

#: ../../client/solo/commands/query.py:278
msgid "show the direct dependencies graph for the given package"
msgstr ""

#: ../../client/solo/commands/query.py:285
#: ../../client/solo/commands/query.py:302
msgid ""
"include system packages, build-time dependencies and circular dependencies "
"information"
msgstr ""

#: ../../client/solo/commands/query.py:295
msgid "show the inverse dependencies graph for the given package"
msgstr ""

#: ../../client/solo/commands/query.py:348
msgid "Belong Search"
msgstr "Поиск принадлежностей"

#: ../../client/solo/commands/query.py:400
#: ../../client/solo/commands/query.py:557
#: ../../client/solo/commands/query.py:651
#: ../../client/solo/commands/query.py:765
#: ../../client/solo/commands/query.py:955
#: ../../client/solo/commands/query.py:1324
#: ../../client/solo/commands/query.py:1383
#: ../../client/solo/commands/query.py:1446
#: ../../client/solo/commands/query.py:1494
msgid "Keyword"
msgstr "Ключевое слово"

#: ../../client/solo/commands/query.py:403
#: ../../client/solo/commands/query.py:572
#: ../../client/solo/commands/query.py:652
#: ../../client/solo/commands/query.py:697
#: ../../client/solo/commands/query.py:768
#: ../../client/solo/commands/query.py:957
#: ../../client/solo/commands/query.py:1022
#: ../../client/solo/commands/query.py:1271
#: ../../client/solo/commands/query.py:1325
#: ../../client/solo/commands/query.py:1386
#: ../../client/solo/commands/query.py:1449
#: ../../client/solo/commands/query.py:1497
#: ../../client/solo/commands/search.py:180
#: ../../client/solo/commands/match.py:205
msgid "Found"
msgstr "Найдено"

#: ../../client/solo/commands/query.py:405
#: ../../client/solo/commands/query.py:570
#: ../../client/solo/commands/query.py:654
#: ../../client/solo/commands/query.py:762
#: ../../client/solo/commands/query.py:952
#: ../../client/solo/commands/query.py:1323
#: ../../client/solo/commands/query.py:1381
#: ../../client/solo/commands/query.py:1444
#: ../../client/solo/commands/query.py:1492
#: ../../client/solo/commands/search.py:182
#: ../../client/solo/commands/match.py:207
msgid "entry"
msgid_plural "entries"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../client/solo/commands/query.py:422
msgid "ChangeLog Search"
msgstr "Поиск логов изменений"

#: ../../client/solo/commands/query.py:432
#: ../../client/solo/commands/query.py:440
#: ../../client/solo/commands/_manage.py:386
msgid "No match for"
msgstr "Ничего не найдено для"

#: ../../client/solo/commands/query.py:464
msgid "No ChangeLog available"
msgstr "Недоступно ни одного лога изменений"

#: ../../client/solo/commands/query.py:479
msgid "Attention: developer-repo option not enabled"
msgstr ""

#: ../../client/solo/commands/query.py:499
msgid "Reverse Dependencies Search"
msgstr ""

#: ../../client/solo/commands/query.py:559
#: ../../client/solo/commands/libtest.py:195
msgid "Matched"
msgstr "Подсчитано"

#: ../../client/solo/commands/query.py:565
msgid "from repository"
msgstr "из репозитория"

#: ../../client/solo/commands/query.py:567
msgid "from the installed packages repository"
msgstr ""

#: ../../client/solo/commands/query.py:595
msgid "Description Search"
msgstr "Поиск описания"

#: ../../client/solo/commands/query.py:619
#: ../../client/solo/commands/query.py:775
#: ../../client/solo/commands/query.py:834
#: ../../client/solo/commands/query.py:966
#: ../../client/solo/commands/query.py:1331
#: ../../client/solo/commands/query.py:1394
#: ../../client/solo/commands/query.py:1457
#: ../../client/solo/commands/query.py:1505
msgid "No matches"
msgstr "Ничего не найдено"

#: ../../client/solo/commands/query.py:670
msgid "Files Search"
msgstr ""

#: ../../client/solo/commands/query.py:698
msgid "files"
msgstr "файлы"

#: ../../client/solo/commands/query.py:726
msgid "License Search"
msgstr "Поиск лицензии"

#: ../../client/solo/commands/query.py:792
msgid "Listing Packages"
msgstr ""

#: ../../client/solo/commands/query.py:885
msgid "Repository is not available"
msgstr ""

#: ../../client/solo/commands/query.py:916
msgid "Searching Mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:998
msgid "Needed Packages Search"
msgstr ""

#: ../../client/solo/commands/query.py:1023
msgid "libraries"
msgstr "библиотеки"

#: ../../client/solo/commands/query.py:1039
msgid "Orphans Search"
msgstr "Поиск осиротевших файлов"

#: ../../client/solo/commands/query.py:1107
msgid "Analyzing"
msgstr "Анализ"

#: ../../client/solo/commands/query.py:1121
msgid "Analyzed directories"
msgstr "Проанализированные директории"

#: ../../client/solo/commands/query.py:1126
msgid "Masked directories"
msgstr "Замаскированные директории"

#: ../../client/solo/commands/query.py:1131
msgid "Number of files collected on the filesystem"
msgstr "Число файлов, собранных в файловой системе"

#: ../../client/solo/commands/query.py:1136
msgid "Now searching among installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:1167
msgid "Checking"
msgstr ""

#: ../../client/solo/commands/query.py:1182
msgid "Number of total files"
msgstr "Всего файлов"

#: ../../client/solo/commands/query.py:1188
msgid "Number of matching files"
msgstr "Число подсчитанных файлов"

#: ../../client/solo/commands/query.py:1195
msgid "Number of orphaned files"
msgstr "Число осиротевших файлов"

#: ../../client/solo/commands/query.py:1201
msgid "Writing file to disk"
msgstr "Запись файла на диск"

#: ../../client/solo/commands/query.py:1231
msgid "Total space wasted"
msgstr ""

#: ../../client/solo/commands/query.py:1249
msgid "Required Packages Search"
msgstr ""

#: ../../client/solo/commands/query.py:1267
msgid "Library"
msgstr "Библиотека"

#: ../../client/solo/commands/query.py:1291
msgid "Package Set Search"
msgstr "Поиск набора пакетов"

#: ../../client/solo/commands/query.py:1347
msgid "Slot Search"
msgstr "Поиск слота"

#: ../../client/solo/commands/query.py:1410
msgid "Tag Search"
msgstr "Поиск тэга"

#: ../../client/solo/commands/query.py:1473
msgid "Revision Search"
msgstr ""

#: ../../client/solo/commands/config.py:68
#: ../../client/solo/commands/security.py:93
#: ../../client/solo/commands/security.py:163
#: ../../client/solo/commands/remove.py:72
#: ../../client/solo/commands/source.py:67
#: ../../client/solo/commands/install.py:75
#: ../../client/solo/commands/upgrade.py:70
#: ../../client/solo/commands/rescue.py:85
#: ../../client/solo/commands/download.py:67
msgid "show what would be done"
msgstr ""

#: ../../client/solo/commands/config.py:74
#: ../../client/solo/commands/remove.py:78
#: ../../client/solo/commands/search.py:76
#: ../../client/solo/commands/command.py:90
#: ../../client/solo/commands/source.py:73
#: ../../client/solo/commands/match.py:89
#: ../../client/solo/commands/install.py:81
#: ../../client/solo/commands/upgrade.py:76
#: ../../client/solo/commands/download.py:73
msgid "verbose output"
msgstr ""

#: ../../client/solo/commands/config.py:122
#: ../../client/solo/commands/status.py:130
#: ../../client/solo/commands/status.py:137
#: ../../client/solo/commands/status.py:144
#: ../../client/solo/commands/remove.py:203
#: ../../client/solo/commands/remove.py:267
#: ../../client/solo/commands/_manage.py:153
#: ../../client/solo/commands/_manage.py:164
#: ../../client/solo/commands/install.py:238
#: ../../client/solo/commands/install.py:264
#: ../../client/solo/commands/install.py:350
msgid "Not available"
msgstr "Недоступно"

#: ../../client/solo/commands/config.py:126
msgid "installed from"
msgstr ""

#: ../../client/solo/commands/config.py:137
#: ../../client/solo/commands/remove.py:290
#: ../../client/solo/commands/_manage.py:230
msgid "Packages involved"
msgstr "Включённые пакеты"

#: ../../client/solo/commands/config.py:161
msgid "configure"
msgstr ""

#: ../../client/solo/commands/config.py:183
msgid "configure installed packages"
msgstr ""

#: ../../client/solo/commands/status.py:105
msgid "Repository name"
msgstr "Имя репозитория"

#: ../../client/solo/commands/status.py:114
msgid "Revision"
msgstr ""

#: ../../client/solo/commands/status.py:126
msgid "Repository class"
msgstr ""

#: ../../client/solo/commands/status.py:133
msgid "GPG"
msgstr ""

#: ../../client/solo/commands/status.py:140
msgid "Notice Board"
msgstr ""

#: ../../client/solo/commands/status.py:147
msgid "Path"
msgstr ""

#: ../../client/solo/commands/status.py:160
msgid "Package URLs"
msgstr ""

#: ../../client/solo/commands/status.py:175
msgid "show Repositories status"
msgstr ""

#: ../../client/solo/commands/update.py:64
#: ../../client/solo/commands/ugc.py:234
#: ../../client/solo/commands/notice.py:57
msgid "repository"
msgstr "репозиторий"

#: ../../client/solo/commands/update.py:67
msgid "force update"
msgstr ""

#: ../../client/solo/commands/update.py:103
msgid "You must be either root or in this group:"
msgstr ""

#: ../../client/solo/commands/update.py:122
msgid "Sending the update request to Entropy Services"
msgstr ""

#: ../../client/solo/commands/update.py:123
msgid "Repositories will be updated in background"
msgstr ""

#: ../../client/solo/commands/update.py:132
msgid "app-admin/rigo-daemon not installed. Update not allowed."
msgstr ""

#: ../../client/solo/commands/update.py:191
msgid "Have a nice day"
msgstr ""

#: ../../client/solo/commands/update.py:196
msgid "Repositories update not allowed"
msgstr ""

#: ../../client/solo/commands/update.py:221
#: ../../client/solo/commands/hop.py:116
msgid "No repositories specified in"
msgstr "Ни одного репозитория не отмечено в"

#: ../../client/solo/commands/update.py:260
#: ../../client/solo/commands/notice.py:123
msgid "Notice board not available"
msgstr "Доска уведомлений недоступна"

#: ../../client/solo/commands/update.py:274
msgid "Notice board"
msgstr "Доска уведомлений"

#. title
#: ../../client/solo/commands/update.py:287
#: ../../client/solo/commands/security.py:215
#: ../../client/solo/commands/ugc.py:355 ../../client/solo/commands/ugc.py:614
#: ../../client/solo/commands/notice.py:146
#: ../../client/solo/commands/notice.py:183
msgid "Title"
msgstr "Заголовок"

#: ../../client/solo/commands/update.py:296
msgid "update repositories"
msgstr ""

#: ../../client/solo/commands/security.py:68
#: ../../client/solo/commands/security.py:627
msgid "system security tools"
msgstr ""

#: ../../client/solo/commands/security.py:73
msgid "verify installed files using stored checksums"
msgstr ""

#: ../../client/solo/commands/security.py:77
msgid "consider mtime instead of SHA256 (false positives ahead)"
msgstr ""

#: ../../client/solo/commands/security.py:81
msgid "update hashes and mtime (useful after editing config files)"
msgstr ""

#: ../../client/solo/commands/security.py:85
msgid "reinstall faulty packages"
msgstr ""

#: ../../client/solo/commands/security.py:96
#: ../../client/solo/commands/security.py:166
#: ../../client/solo/commands/install.py:89
#: ../../client/solo/commands/upgrade.py:84
msgid "just download packages"
msgstr "просто загрузить пакеты"

#: ../../client/solo/commands/security.py:110
msgid "download the latest Security Advisories"
msgstr "загружает последние уведомления о безопасности"

#: ../../client/solo/commands/security.py:114
msgid "force download"
msgstr ""

#: ../../client/solo/commands/security.py:123
msgid "list all the available Security Advisories"
msgstr "выдаёт список всех доступных уведомлений о безопасности"

#: ../../client/solo/commands/security.py:129
msgid "list only affected"
msgstr "показать только список уязвимого"

#: ../../client/solo/commands/security.py:132
msgid "list only unaffected"
msgstr "показать только список неуязвимого"

#: ../../client/solo/commands/security.py:141
msgid "show information about provided advisories identifiers"
msgstr "показать информацию о идентификаторах предоставленных уведомлений"

#: ../../client/solo/commands/security.py:146
msgid "advistory indentifier"
msgstr ""

#: ../../client/solo/commands/security.py:153
msgid "automatically install all the available security updates"
msgstr "автоматически установить все доступные обновления безопасности"

#: ../../client/solo/commands/security.py:211
msgid "Advisory Identifier"
msgstr ""

#: ../../client/solo/commands/security.py:243
msgid "Background"
msgstr "Фон"

#: ../../client/solo/commands/security.py:250
msgid "Exploitable"
msgstr "Добываемое"

#: ../../client/solo/commands/security.py:256
msgid "Impact"
msgstr "Влияние"

#: ../../client/solo/commands/security.py:263
msgid "Impact type"
msgstr "Тип влияния"

#: ../../client/solo/commands/security.py:268
msgid "Revised"
msgstr "Проверено"

#: ../../client/solo/commands/security.py:273
msgid "Announced"
msgstr "Объявлено"

#: ../../client/solo/commands/security.py:278
msgid "Synopsis"
msgstr "Перспектива"

#: ../../client/solo/commands/security.py:285
msgid "References"
msgstr "Ссылки"

#: ../../client/solo/commands/security.py:291
msgid "Upstream bugs"
msgstr "Баги в процессе"

#: ../../client/solo/commands/security.py:297
msgid "Affected"
msgstr "Уязвимо"

#: ../../client/solo/commands/security.py:305
msgid "vulnerable versions"
msgstr "уязвимые версии"

#: ../../client/solo/commands/security.py:308
msgid "unaffected versions"
msgstr "неуязвимые версии"

#: ../../client/solo/commands/security.py:313
msgid "Workaround"
msgstr "Оптимизация"

#: ../../client/solo/commands/security.py:320
msgid "Resolution"
msgstr "Разрешение"

#: ../../client/solo/commands/security.py:347
msgid "Checking system files"
msgstr ""

#: ../../client/solo/commands/security.py:375
msgid "no checksum information"
msgstr ""

#: ../../client/solo/commands/security.py:424
msgid "found altered files"
msgstr ""

#: ../../client/solo/commands/security.py:443
msgid "assimilated new hashes and mtime"
msgstr ""

#: ../../client/solo/commands/security.py:454
msgid "unavailable"
msgstr ""

#: ../../client/solo/commands/security.py:461
msgid "No altered files found"
msgstr ""

#: ../../client/solo/commands/security.py:487
msgid "Altered files have been found"
msgstr ""

#: ../../client/solo/commands/security.py:491
msgid "Packages have been reinstalled successfully"
msgstr ""

#: ../../client/solo/commands/security.py:523
msgid "No advisories available or applicable"
msgstr "Нет доступных либо применимых уведомлений"

#: ../../client/solo/commands/security.py:566
msgid "Advisory does not exist"
msgstr ""

#: ../../client/solo/commands/security.py:592
msgid "Calculating security updates"
msgstr "Подсчёт обновлений безопасности"

#: ../../client/solo/commands/security.py:611
msgid "All the available updates have been already installed"
msgstr "Все возможные обновления уже были установлены"

#: ../../client/solo/commands/moo.py:114
msgid "moo at user"
msgstr ""

#: ../../client/solo/commands/moo.py:121
msgid "bow to lxnay"
msgstr ""

#: ../../client/solo/commands/remove.py:82
#: ../../client/solo/commands/source.py:81
#: ../../client/solo/commands/install.py:98
#: ../../client/solo/commands/upgrade.py:93
#: ../../client/solo/commands/download.py:81
msgid "exclude package dependencies"
msgstr ""

#: ../../client/solo/commands/remove.py:86
#: ../../client/solo/commands/source.py:85
#: ../../client/solo/commands/install.py:102
#: ../../client/solo/commands/upgrade.py:97
#: ../../client/solo/commands/download.py:85
msgid "do not calculate dependencies recursively"
msgstr ""

#: ../../client/solo/commands/remove.py:90
#: ../../client/solo/commands/source.py:90
#: ../../client/solo/commands/install.py:106
#: ../../client/solo/commands/upgrade.py:101
#: ../../client/solo/commands/download.py:90
msgid "include dependencies no longer needed"
msgstr ""

#: ../../client/solo/commands/remove.py:94
#: ../../client/solo/commands/install.py:110
#: ../../client/solo/commands/upgrade.py:105
msgid "when used with --deep, include virtual packages"
msgstr ""

#: ../../client/solo/commands/remove.py:98
#: ../../client/solo/commands/install.py:114
#: ../../client/solo/commands/upgrade.py:109
msgid "remove package configuration files no longer needed"
msgstr ""

#: ../../client/solo/commands/remove.py:102
msgid "force system packages removal (dangerous!)"
msgstr ""

#: ../../client/solo/commands/remove.py:156
msgid "removal"
msgstr ""

#: ../../client/solo/commands/remove.py:172
#: ../../client/solo/commands/source.py:153
#: ../../client/solo/commands/download.py:155
msgid "All done"
msgstr "Всё сделано"

#: ../../client/solo/commands/remove.py:184
msgid "These are the packages that should be MANUALLY removed"
msgstr ""

#: ../../client/solo/commands/remove.py:190
#: ../../client/solo/commands/install.py:330
msgid "These are the packages that would be removed"
msgstr "Здесь перечислены пакеты, которые должны быть удалены"

#: ../../client/solo/commands/remove.py:240
msgid "These are the chosen packages"
msgstr "Вот выбранные пакеты"

#: ../../client/solo/commands/remove.py:253
msgid "vital package"
msgstr "жизненно важный пакет"

#: ../../client/solo/commands/remove.py:254
msgid "Removal forbidden"
msgstr "Удаление запрещено"

#: ../../client/solo/commands/remove.py:304
msgid "Packages that would be removed"
msgstr ""

#: ../../client/solo/commands/remove.py:334
#: ../../client/solo/commands/install.py:421
msgid "Freed disk space"
msgstr "Свободное место на диске"

#: ../../client/solo/commands/remove.py:341
msgid "Total bandwidth wasted"
msgstr ""

#: ../../client/solo/commands/remove.py:371
#: ../../client/solo/commands/rescue.py:629
msgid "Nothing to do"
msgstr "Делать нечего"

#: ../../client/solo/commands/remove.py:378
msgid "Would you like to calculate dependencies ?"
msgstr "Желаете ли подсчитать зависимости?"

#: ../../client/solo/commands/remove.py:385
msgid "Would you like to remove them now ?"
msgstr "Вы желаете удалить их сейчас ?"

#: ../../client/solo/commands/remove.py:406
msgid "Ouch!"
msgstr ""

#: ../../client/solo/commands/remove.py:407
msgid "the following system packages were pulled in"
msgstr ""

#: ../../client/solo/commands/remove.py:431
msgid "Would you like to proceed ?"
msgstr "Желаете продолжить ?"

#: ../../client/solo/commands/remove.py:446
msgid "remove packages from system"
msgstr ""

#: ../../client/solo/commands/search.py:68
msgid "search keyword"
msgstr ""

#: ../../client/solo/commands/search.py:72
#: ../../client/solo/commands/match.py:85
msgid "quiet output, for scripting purposes"
msgstr ""

#: ../../client/solo/commands/search.py:81
msgid "search among installed packages only"
msgstr ""

#: ../../client/solo/commands/search.py:85
msgid "search among available packages only"
msgstr ""

#: ../../client/solo/commands/search.py:167
msgid "Searching"
msgstr "Поиск"

#: ../../client/solo/commands/search.py:214
msgid "search packages in repositories"
msgstr "найти пакеты в репозиториях"

#: ../../client/solo/commands/deptest.py:64
#: ../../client/solo/commands/libtest.py:66
#: ../../client/solo/commands/unused.py:65
msgid "show less details (useful for scripting)"
msgstr "показать меньше деталей (полезно для скриптинга)"

#: ../../client/solo/commands/deptest.py:67
#: ../../client/solo/commands/libtest.py:72 ../../services/kernel-switcher:313
msgid "just show what would be done"
msgstr "показывает только то, что будет сделано"

#: ../../client/solo/commands/deptest.py:102
msgid "Running dependency test"
msgstr "Запуск теста зависимостей"

#: ../../client/solo/commands/deptest.py:157
msgid "These are the dependencies not found"
msgstr "Имеются ненайденные зависимости"

#: ../../client/solo/commands/deptest.py:167
msgid "Needed by"
msgstr "Необходимо для"

#: ../../client/solo/commands/deptest.py:178
msgid "Would you like to install the packages ?"
msgstr ""

#: ../../client/solo/commands/deptest.py:185
#: ../../client/solo/commands/libtest.py:224
msgid "Installing available packages in"
msgstr "Установка доступных пакетов в"

#: ../../client/solo/commands/deptest.py:186
#: ../../client/solo/commands/libtest.py:225
msgid "10 seconds"
msgstr "10 секунд"

#: ../../client/solo/commands/deptest.py:204
msgid "look for unsatisfied dependencies"
msgstr "ищет неудовлетворенные зависимости"

#: ../../client/solo/commands/command.py:70
msgid "not a valid directory"
msgstr ""

#: ../../client/solo/commands/command.py:93
#: ../../client/solo/commands/source.py:77
#: ../../client/solo/commands/install.py:85
#: ../../client/solo/commands/upgrade.py:80
#: ../../client/solo/commands/download.py:77
msgid "quiet output"
msgstr ""

#: ../../client/solo/commands/command.py:319
msgid "Acquiring Entropy Resources Lock, please wait..."
msgstr ""

#: ../../client/solo/commands/command.py:326
#: ../../client/solo/commands/command.py:356
#: ../../services/kernel-switcher:159 ../../services/kernel-switcher:271
msgid "Another Entropy is currently running."
msgstr ""

#: ../../client/solo/commands/command.py:384
msgid "When you wrote"
msgstr "Как Вы написали"

#: ../../client/solo/commands/command.py:386
msgid "You Meant(tm)"
msgstr "А ты думал (TM)"

#: ../../client/solo/commands/command.py:387
msgid "one of these below?"
msgstr "один из нижеуказанных?"

#: ../../client/solo/commands/cache.py:60
msgid "manage cache"
msgstr ""

#: ../../client/solo/commands/cache.py:64
msgid "clean Entropy Library Cache"
msgstr ""

#: ../../client/solo/commands/cache.py:67
msgid "show more details"
msgstr "показать больше деталей"

#: ../../client/solo/commands/cache.py:70
msgid "print results in a scriptable way"
msgstr "отображает результаты через скрипты"

#: ../../client/solo/commands/cache.py:122
msgid "Cleaning Entropy cache, please wait ..."
msgstr ""

#: ../../client/solo/commands/cache.py:129
msgid "Entropy cache cleaned."
msgstr ""

#: ../../client/solo/commands/cache.py:140
msgid "manage Entropy Library Cache"
msgstr ""

#: ../../client/solo/commands/ugc.py:71 ../../client/solo/commands/ugc.py:1005
msgid "manage User Generated Content"
msgstr ""

#: ../../client/solo/commands/ugc.py:75
msgid "login against the given repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:77 ../../client/solo/commands/ugc.py:87
#: ../../client/solo/commands/ugc.py:99 ../../client/solo/commands/repo.py:72
#: ../../client/solo/commands/repo.py:80 ../../client/solo/commands/repo.py:96
#: ../../client/solo/commands/repo.py:109
msgid "repository name"
msgstr ""

#: ../../client/solo/commands/ugc.py:80 ../../client/solo/commands/ugc.py:90
msgid "force action"
msgstr "принудительно запустить действие"

#: ../../client/solo/commands/ugc.py:85
msgid "logout from the given repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:96
msgid ""
"manage package documents in the selected repository (comments, files, videos)"
msgstr ""

#: ../../client/solo/commands/ugc.py:104
msgid "manage Documents"
msgstr ""

#: ../../client/solo/commands/ugc.py:108
msgid "get available documents for the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:111 ../../client/solo/commands/ugc.py:119
#: ../../client/solo/commands/ugc.py:127 ../../client/solo/commands/ugc.py:149
#: ../../client/solo/commands/ugc.py:156
msgid "package name (example: x11-libs/qt)"
msgstr ""

#: ../../client/solo/commands/ugc.py:116
msgid "add a new document to the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:124
msgid "remove documents from database using their identifiers"
msgstr "удалить документы из базы данных, используя их наименования"

#: ../../client/solo/commands/ugc.py:130
msgid "document identifier"
msgstr ""

#: ../../client/solo/commands/ugc.py:137
msgid "manage package votes in the selected repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:143
msgid "manage Votes"
msgstr ""

#: ../../client/solo/commands/ugc.py:147
msgid "get vote for the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:154
msgid "add vote to the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:194 ../../client/solo/commands/ugc.py:298
msgid "Invalid repository"
msgstr "Некорректный репозиторий"

#: ../../client/solo/commands/ugc.py:205 ../../client/solo/commands/ugc.py:309
#: ../../client/solo/commands/ugc.py:433 ../../client/solo/commands/ugc.py:515
#: ../../client/solo/commands/ugc.py:731 ../../client/solo/commands/ugc.py:848
#: ../../client/solo/commands/ugc.py:886
msgid "Repository does not support Entropy Services."
msgstr ""

#: ../../client/solo/commands/ugc.py:215
msgid "Already logged in as"
msgstr "Уже вошли в систему как"

#: ../../client/solo/commands/ugc.py:217
msgid "Please logout first"
msgstr "Пожалуйста, сначала выйдите из системы."

#: ../../client/solo/commands/ugc.py:229
msgid "Username"
msgstr "Имя пользователя"

#: ../../client/solo/commands/ugc.py:230
msgid "Password"
msgstr "Пароль"

#: ../../client/solo/commands/ugc.py:234
msgid "Please login against"
msgstr "Пожалуйста, залогиньтесь снова"

#: ../../client/solo/commands/ugc.py:242
msgid "Login aborted. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:256
msgid "Authentication error. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:265
msgid "Communication error. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:275
msgid "Successfully logged in as"
msgstr "Успешно вошли в систему как"

#: ../../client/solo/commands/ugc.py:282
msgid "From now on, any UGC action will be committed as this user"
msgstr ""
"С текущего момента любое действие UGC будет определено как от этого "
"пользователя"

#: ../../client/solo/commands/ugc.py:319
msgid "Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:328
msgid "User"
msgstr "Пользователь"

#: ../../client/solo/commands/ugc.py:330
msgid "has been logged out."
msgstr ""

#: ../../client/solo/commands/ugc.py:339
msgid "No title"
msgstr "Нет заголовка"

#: ../../client/solo/commands/ugc.py:366
#: ../../client/solo/commands/notice.py:189
msgid "Content"
msgstr "Содержимое"

#: ../../client/solo/commands/ugc.py:383
msgid "no votes"
msgstr ""

#: ../../client/solo/commands/ugc.py:389
msgid "current package vote"
msgstr ""

#: ../../client/solo/commands/ugc.py:450 ../../client/solo/commands/ugc.py:460
#: ../../client/solo/commands/ugc.py:477 ../../client/solo/commands/ugc.py:696
#: ../../client/solo/commands/ugc.py:767 ../../client/solo/commands/ugc.py:792
#: ../../client/solo/commands/ugc.py:804 ../../client/solo/commands/ugc.py:816
#: ../../client/solo/commands/ugc.py:861 ../../client/solo/commands/ugc.py:939
#: ../../client/solo/commands/ugc.py:974 ../../client/solo/commands/ugc.py:985
msgid "UGC error"
msgstr "Ошибка UGC"

#: ../../client/solo/commands/ugc.py:489
msgid "Number of downloads"
msgstr "Число загрузок"

#: ../../client/solo/commands/ugc.py:496
msgid "No User Generated Content available."
msgstr ""

#: ../../client/solo/commands/ugc.py:527 ../../client/solo/commands/ugc.py:898
msgid "Not logged in, please login first."
msgstr ""

#: ../../client/solo/commands/ugc.py:537
msgid "Add document"
msgstr "Добавить документ"

#: ../../client/solo/commands/ugc.py:542
msgid "text comment"
msgstr "текстовый документ"

#: ../../client/solo/commands/ugc.py:543
msgid "icon"
msgstr ""

#: ../../client/solo/commands/ugc.py:544
msgid "simple file"
msgstr "простой файл"

#: ../../client/solo/commands/ugc.py:545
msgid "simple image"
msgstr "простое изображение"

#: ../../client/solo/commands/ugc.py:546
msgid "video"
msgstr ""

#: ../../client/solo/commands/ugc.py:563
msgid "Insert document title"
msgstr "Введите заголовок документа"

#: ../../client/solo/commands/ugc.py:565
msgid "Insert document description/comment"
msgstr "Введите описание/комментарий к документу"

#: ../../client/solo/commands/ugc.py:568
msgid "Insert document's keywords (space separated)"
msgstr "Введите ключевые слова документа (разделяя их пробелом)"

#: ../../client/solo/commands/ugc.py:570
msgid "Choose document type"
msgstr "Выберите тип документа"

#: ../../client/solo/commands/ugc.py:575 ../../client/solo/commands/ugc.py:595
msgid "Entropy UGC document submission"
msgstr "Представление документов Entropy UGC"

#: ../../client/solo/commands/ugc.py:592
msgid "Insert document path"
msgstr "Введите путь к документу"

#: ../../client/solo/commands/ugc.py:608 ../../client/solo/commands/ugc.py:748
#: ../../client/solo/commands/ugc.py:951
msgid "Please review your submission"
msgstr "Пожалуйста, оставьте своё мнение"

#: ../../client/solo/commands/ugc.py:636
msgid "Document path"
msgstr "Путь к документу"

#: ../../client/solo/commands/ugc.py:643
msgid "Document type"
msgstr "Тип документа"

#: ../../client/solo/commands/ugc.py:649 ../../client/solo/commands/ugc.py:963
msgid "Do you want to submit?"
msgstr ""

#: ../../client/solo/commands/ugc.py:681
msgid "Invalid document"
msgstr ""

#: ../../client/solo/commands/ugc.py:710
msgid "Document added, thank you!"
msgstr "Документ добавлен, спасибо!"

#: ../../client/solo/commands/ugc.py:741
msgid "Documents removal"
msgstr "Удаление документов"

#: ../../client/solo/commands/ugc.py:754
msgid "Document identifiers"
msgstr "Имена документов"

#: ../../client/solo/commands/ugc.py:759
msgid "Would you like to review them?"
msgstr ""

#: ../../client/solo/commands/ugc.py:782
msgid "Would you like to continue with the removal?"
msgstr ""

#: ../../client/solo/commands/ugc.py:805
msgid "cannot get the requested Document"
msgstr ""

#: ../../client/solo/commands/ugc.py:825
msgid "UGC status"
msgstr "Статус UGC"

#: ../../client/solo/commands/ugc.py:826
msgid "removed successfully"
msgstr ""

#: ../../client/solo/commands/ugc.py:907
msgid "add vote"
msgstr ""

#: ../../client/solo/commands/ugc.py:915
msgid "Insert your vote (from 1 to 5)"
msgstr "Введите Ваш голос (от 1 до 5)"

#: ../../client/solo/commands/ugc.py:919
msgid "Entropy UGC vote submission"
msgstr "Подчинение оценки Entropy UGC"

#: ../../client/solo/commands/ugc.py:940
msgid "invalid vote, must be in range"
msgstr ""

#: ../../client/solo/commands/ugc.py:957
msgid "Vote"
msgstr "Проголосовать"

#: ../../client/solo/commands/ugc.py:986
msgid "already voted"
msgstr ""

#: ../../client/solo/commands/ugc.py:994
msgid "vote added, thank you!"
msgstr ""

#: ../../client/solo/commands/help.py:70
msgid "Entropy Command Line Client, Equo"
msgstr ""

#: ../../client/solo/commands/help.py:97 ../../services/kernel-switcher:337
msgid "this help"
msgstr ""

#: ../../client/solo/commands/repo.py:65
#: ../../client/solo/commands/repo.py:639
msgid "manage repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:69
msgid "enable repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:77
msgid "disable repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:85
msgid "add a repository"
msgstr ""

#: ../../client/solo/commands/repo.py:88
msgid "repository string"
msgstr ""

#: ../../client/solo/commands/repo.py:93
msgid "remove a repository"
msgstr ""

#: ../../client/solo/commands/repo.py:101
msgid "list active repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:106
msgid "reorder mirrors basing on response time"
msgstr ""

#: ../../client/solo/commands/repo.py:112
msgid "simulate execution"
msgstr ""

#: ../../client/solo/commands/repo.py:117
msgid "merge content of source repository to destination"
msgstr ""

#: ../../client/solo/commands/repo.py:119
msgid "source repository"
msgstr ""

#: ../../client/solo/commands/repo.py:121
msgid "destination repository"
msgstr ""

#: ../../client/solo/commands/repo.py:125
msgid "also remove dependency conflicts during merge"
msgstr ""

#: ../../client/solo/commands/repo.py:218
msgid "repository already enabled"
msgstr ""

#: ../../client/solo/commands/repo.py:227
#: ../../client/solo/commands/repo.py:281
#: ../../client/solo/commands/repo.py:502
msgid "repository not available"
msgstr ""

#: ../../client/solo/commands/repo.py:247
msgid "repository enabled"
msgstr ""

#: ../../client/solo/commands/repo.py:253
msgid "cannot enable repository"
msgstr ""

#: ../../client/solo/commands/repo.py:272
msgid "repository already disabled"
msgstr ""

#: ../../client/solo/commands/repo.py:303
#: ../../client/solo/commands/repo.py:317
msgid "cannot disable repository"
msgstr ""

#: ../../client/solo/commands/repo.py:311
msgid "repository disabled"
msgstr ""

#: ../../client/solo/commands/repo.py:333
msgid "invalid data, skipping"
msgstr ""

#: ../../client/solo/commands/repo.py:355
msgid "Adding repository string"
msgstr ""

#: ../../client/solo/commands/repo.py:365
msgid "invalid repository string"
msgstr ""

#: ../../client/solo/commands/repo.py:373
msgid "Repository id:"
msgstr ""

#: ../../client/solo/commands/repo.py:376
msgid "Description:"
msgstr ""

#: ../../client/solo/commands/repo.py:379
msgid "Repository format:"
msgstr ""

#: ../../client/solo/commands/repo.py:383
msgid "Packages URL:"
msgstr ""

#: ../../client/solo/commands/repo.py:386
msgid "None given"
msgstr ""

#: ../../client/solo/commands/repo.py:388
msgid "Repository URL:"
msgstr ""

#: ../../client/solo/commands/repo.py:397
msgid "repository added succesfully"
msgstr ""

#: ../../client/solo/commands/repo.py:402
msgid "cannot add repository"
msgstr ""

#: ../../client/solo/commands/repo.py:424
msgid "repository id not available"
msgstr ""

#: ../../client/solo/commands/repo.py:444
msgid "repository removed succesfully"
msgstr ""

#: ../../client/solo/commands/repo.py:450
msgid "cannot remove repository"
msgstr ""

#: ../../client/solo/commands/repo.py:516
msgid "mirror order:"
msgstr ""

#: ../../client/solo/commands/repo.py:526
msgid "mirrors sorted successfully"
msgstr ""

#: ../../client/solo/commands/repo.py:545
#: ../../client/solo/commands/repo.py:552
msgid "repository not found"
msgstr ""

#: ../../client/solo/commands/repo.py:561
msgid "repository cannot be source and destination"
msgstr ""

#: ../../client/solo/commands/repo.py:569
msgid "merging repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:585
msgid "working on repository"
msgstr ""

#: ../../client/solo/commands/repo.py:602
msgid "merging package"
msgstr ""

#: ../../client/solo/commands/repo.py:626
msgid "done merging packages"
msgstr ""

#: ../../client/solo/commands/source.py:94
#: ../../client/solo/commands/download.py:94
msgid "calculate dependencies relaxing constraints"
msgstr ""

#: ../../client/solo/commands/source.py:98
#: ../../client/solo/commands/install.py:94
#: ../../client/solo/commands/upgrade.py:89
#: ../../client/solo/commands/download.py:98
msgid "include build-time dependencies"
msgstr ""

#: ../../client/solo/commands/source.py:102
msgid "save files into the current working directory"
msgstr ""

#: ../../client/solo/commands/source.py:138
msgid "Source code download"
msgstr ""

#: ../../client/solo/commands/source.py:175
msgid "sources download"
msgstr ""

#: ../../client/solo/commands/source.py:197
msgid "download packages source code"
msgstr ""

#: ../../client/solo/commands/yell.py:34
msgid "You should run equo --help"
msgstr "Вы должны запустить equo --help"

#: ../../client/solo/commands/yell.py:35
msgid "You didn't run equo --help, did you?"
msgstr "Вы не запустили equo --help, так ведь?"

#: ../../client/solo/commands/yell.py:36
msgid "Did you even read equo --help??"
msgstr "Вы когда-нибудь читали equo --help??"

#: ../../client/solo/commands/yell.py:37
msgid "I give up. Run that equo --help !!!!!!!"
msgstr "Я умываю руки. Запусти уже этот чёртов equo --help !!!!!!!"

#: ../../client/solo/commands/yell.py:38
msgid "OH MY GOD. RUN equo --heeeeeeeeeeeeeelp"
msgstr "О, Боже. ЗАПУСТИ ЭТОТ ГРЁБАНЫЙ equo --help..."

#: ../../client/solo/commands/yell.py:39
msgid "Illiteracy is a huge problem in this world"
msgstr "Невежество - путь к ереси."

#: ../../client/solo/commands/yell.py:40
msgid "Ok i give up, you are hopeless"
msgstr "Окей, я сдаюсь. Ты неисправим."

#: ../../client/solo/commands/yell.py:41 ../../client/solo/commands/yell.py:42
#: ../../client/solo/commands/yell.py:43 ../../client/solo/commands/yell.py:44
#: ../../client/solo/commands/yell.py:45 ../../client/solo/commands/yell.py:47
msgid "Go to hell."
msgstr "Иди к чёрту."

#: ../../client/solo/commands/yell.py:46
msgid "Stop that, you idiot."
msgstr ""

#: ../../client/solo/commands/yell.py:117
msgid "yell at user"
msgstr ""

#: ../../client/solo/commands/libtest.py:77
msgid "print broken files to stdout"
msgstr "записывает испорченные файлы в stdout"

#: ../../client/solo/commands/libtest.py:82
msgid "dump results to files"
msgstr "записать результаты в файлы"

#: ../../client/solo/commands/libtest.py:137
#: ../../client/solo/commands/libtest.py:170
msgid "System is healthy"
msgstr "В системе все в порядке"

#: ../../client/solo/commands/libtest.py:176
msgid "Libraries/Executables statistics"
msgstr "Статистика библиотек/исполняемых файлов"

#: ../../client/solo/commands/libtest.py:182
msgid "Not matched"
msgstr "Не подсчитано"

#: ../../client/solo/commands/libtest.py:218
msgid "Would you like to install them ?"
msgstr "Вы желаете их инсталлировать ?"

#: ../../client/solo/commands/libtest.py:246
msgid "look for missing libraries"
msgstr "ищет испорченные библиотеки"

#: ../../client/solo/commands/notice.py:153
#: ../../client/solo/commands/conf.py:221
#: ../../magneto/src/magneto/core/interfaces.py:175
msgid "Exit"
msgstr "Выход"

#: ../../client/solo/commands/notice.py:162
msgid "Choose one by typing its identifier"
msgstr "Выберите один, набрав его название"

#: ../../client/solo/commands/notice.py:194
msgid "Link"
msgstr "Ссылка"

#: ../../client/solo/commands/notice.py:203
msgid "Press Enter to continue"
msgstr "Нажмите Enter, чтобы продолжить"

#: ../../client/solo/commands/notice.py:211
msgid "repository notice board reader"
msgstr "просмотрщик доски уведомлений репозитория"

#: ../../client/solo/commands/_manage.py:90
msgid "Scanning configuration files to update"
msgstr ""

#: ../../client/solo/commands/_manage.py:99
msgid "No configuration files to update."
msgstr ""

#: ../../client/solo/commands/_manage.py:104
#, python-format
msgid "There is %s configuration file needing update"
msgid_plural "There are %s configuration files needing update"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../client/solo/commands/_manage.py:112
msgid "Please run"
msgstr "Будьте добры запустить"

#: ../../client/solo/commands/_manage.py:128
msgid "These are the selected packages"
msgstr ""

#: ../../client/solo/commands/_manage.py:181
msgid "Versions"
msgstr "Версии"

#: ../../client/solo/commands/_manage.py:205
#: ../../client/solo/commands/_manage.py:211
msgid "Reinstall"
msgstr "Переустановить"

#: ../../client/solo/commands/_manage.py:206
msgid "Switch repo"
msgstr "Переключить репозиторий"

#: ../../client/solo/commands/_manage.py:213
msgid "Install"
msgstr "Инсталлировать"

#: ../../client/solo/commands/_manage.py:215
msgid "Upgrade"
msgstr "Обновить"

#: ../../client/solo/commands/_manage.py:217
msgid "Downgrade"
msgstr "Обновить до более старой версии"

#: ../../client/solo/commands/_manage.py:225
msgid "Action"
msgstr "Действие"

#: ../../client/solo/commands/_manage.py:237
msgid "Would you like to continue with the dependencies calculation?"
msgstr ""

#. every package matching app-foo is masked
#: ../../client/solo/commands/_manage.py:339
msgid "Every package matching"
msgstr "Подсчёт каждого пакета"

#: ../../client/solo/commands/_manage.py:341
msgid "is masked"
msgstr "замаскирован"

#: ../../client/solo/commands/_manage.py:359
msgid "Masking reason"
msgstr ""

#: ../../client/solo/commands/_manage.py:371
msgid "matching"
msgstr "подсчитывается"

#: ../../client/solo/commands/_manage.py:373
msgid "is broken"
msgstr "сломан"

#: ../../client/solo/commands/_manage.py:377
msgid "atom"
msgstr "атом"

#: ../../client/solo/commands/_manage.py:379
msgid "in"
msgstr "в"

#: ../../client/solo/commands/_manage.py:388
msgid "in repositories"
msgstr "в репозиториях"

#: ../../client/solo/commands/_manage.py:401
msgid "Not found"
msgstr "Не найдено"

#: ../../client/solo/commands/_manage.py:410
msgid "package needed by"
msgstr ""

#: ../../client/solo/commands/_manage.py:416
#: ../../client/solo/commands/_manage.py:783
#: ../../client/solo/commands/install.py:355
msgid "from"
msgstr "из"

#: ../../client/solo/commands/_manage.py:460
msgid "Skipped"
msgstr "Пропущено"

#: ../../client/solo/commands/_manage.py:481
msgid "is not installed"
msgstr "не установлено"

#: ../../client/solo/commands/_manage.py:510
msgid "Calculating dependencies"
msgstr "Подсчёт зависимостей"

#: ../../client/solo/commands/_manage.py:521
msgid "Dependencies not found"
msgstr ""

#: ../../client/solo/commands/_manage.py:532
msgid "Conflicting packages were pulled in"
msgstr ""

#: ../../client/solo/commands/_manage.py:548
msgid "Please mask conflicts using"
msgstr ""

#: ../../client/solo/commands/_manage.py:597
#: ../../client/solo/commands/_manage.py:638
msgid "download"
msgstr ""

#: ../../client/solo/commands/_manage.py:663
msgid "Repositories are old, please run:"
msgstr ""

#: ../../client/solo/commands/_manage.py:687
msgid "Please update the following critical packages"
msgstr "Пожалуйста, обновитье следующие критические пакеты"

#: ../../client/solo/commands/_manage.py:698
msgid "You should install them as soon as possible"
msgstr "Вы должны установить их как можно раньше"

#: ../../client/solo/commands/_manage.py:719
msgid "Read the license"
msgstr "Читать лицензию"

#: ../../client/solo/commands/_manage.py:724
msgid "Accept the license (I've read it)"
msgstr "Согласиться с лицензией (Я её прочитал(а))"

#: ../../client/solo/commands/_manage.py:729
msgid "Accept the license and don't ask anymore (I've read it)"
msgstr ""
"Согласиться с лицензией и больше об этом не спрашивать (Я её прочитал(а))"

#: ../../client/solo/commands/_manage.py:733
msgid "Quit"
msgstr "Выйти"

#: ../../client/solo/commands/_manage.py:740
msgid "Your choice (type a number and press enter)"
msgstr "Ваш выбор (введите число и нажмите enter)"

#: ../../client/solo/commands/_manage.py:767
msgid "You need to accept the licenses below"
msgstr "Вам нужно согласиться с нижеуказанными лицензиями"

#: ../../client/solo/commands/_manage.py:775
msgid "needed by"
msgstr "необходимо для"

#: ../../client/solo/commands/_manage.py:815
msgid "No license data available"
msgstr ""

#: ../../client/solo/commands/_manage.py:826
msgid "No file viewer"
msgstr "Нет отобразителя файла"

#: ../../client/solo/commands/_manage.py:827
msgid "License saved into"
msgstr "Лицензия сохранена в"

#: ../../client/solo/commands/conf.py:75
msgid "manage configuration file updates"
msgstr ""

#: ../../client/solo/commands/conf.py:79
msgid "update configuration files"
msgstr ""

#: ../../client/solo/commands/conf.py:140
msgid "Scanning filesystem"
msgstr "Сканирование файловой системы"

#: ../../client/solo/commands/conf.py:146
#: ../../client/solo/commands/conf.py:172
msgid "All fine baby. Nothing to do!"
msgstr "Всё пучком. Нечего делать!"

#: ../../client/solo/commands/conf.py:167
msgid "file"
msgstr "файл"

#: ../../client/solo/commands/conf.py:183
#: ../../client/solo/commands/conf.py:628
msgid "Type a number"
msgstr ""

#: ../../client/solo/commands/conf.py:213
msgid "Please choose a file to update by typing its identification number."
msgstr "Выберите файл к обновлению, набрав его идентификационный номер"

#: ../../client/solo/commands/conf.py:217
msgid "Other options are:"
msgstr "Другие опции:"

#: ../../client/solo/commands/conf.py:226
msgid "Automerge all the files asking you one by one"
msgstr "Автоматически объединить файлы, спрашивая Вас о каждом"

#: ../../client/solo/commands/conf.py:231
msgid "Automerge all the files without questioning"
msgstr "Автоматически объединить все файлы без вопросов"

#: ../../client/solo/commands/conf.py:236
msgid "Discard all the files asking you one by one"
msgstr "Пропустить все файлы, спрашивая Вас о каждом"

#: ../../client/solo/commands/conf.py:241
msgid "Discard all the files without questioning"
msgstr "Пропустить все файлы без вопросов"

#: ../../client/solo/commands/conf.py:254
msgid "Please choose an action to take for the selected file."
msgstr "Выберите, что сделать с этим файлом"

#: ../../client/solo/commands/conf.py:259
msgid "Come back to the files list"
msgstr "Обратно в список файлов"

#: ../../client/solo/commands/conf.py:264
msgid "Replace original with update"
msgstr "Заменить оригинал обновлением"

#: ../../client/solo/commands/conf.py:269
msgid "Delete update, keeping original as is"
msgstr "Удалить обновление и оставить оригинал, как было"

#: ../../client/solo/commands/conf.py:274
msgid "Edit proposed file and show diffs again"
msgstr "Редактировать предлагаемый файл и показать различия снова"

#: ../../client/solo/commands/conf.py:279
msgid "Interactively merge original with update"
msgstr ""

#: ../../client/solo/commands/conf.py:284
msgid "Show differences again"
msgstr "Показать различия снова"

#: ../../client/solo/commands/conf.py:380
#: ../../client/solo/commands/conf.py:435
msgid "Source file"
msgstr ""

#: ../../client/solo/commands/conf.py:385
#: ../../client/solo/commands/conf.py:440
msgid "Destination file"
msgstr ""

#: ../../client/solo/commands/conf.py:390
msgid "Overwrite ?"
msgstr "Переписать ?"

#: ../../client/solo/commands/conf.py:399
msgid "Cannot merge"
msgstr ""

#: ../../client/solo/commands/conf.py:445
msgid "Discard ?"
msgstr "Отклонить ?"

#: ../../client/solo/commands/conf.py:451
#: ../../client/solo/commands/conf.py:585
msgid "Discarding"
msgstr "Отклонено"

#: ../../client/solo/commands/conf.py:460
msgid "Cannot remove"
msgstr ""

#: ../../client/solo/commands/conf.py:476
msgid "Editing file"
msgstr "Редактирование файла"

#: ../../client/solo/commands/conf.py:483
msgid "Edited file"
msgstr "Редактированный файл"

#: ../../client/solo/commands/conf.py:485
msgid "showing difference"
msgstr ""

#: ../../client/solo/commands/conf.py:493
#: ../../client/solo/commands/conf.py:597
#: ../../client/solo/commands/conf.py:608
msgid "Automerging"
msgstr ""

#: ../../client/solo/commands/conf.py:533
msgid "Interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:549
msgid "OSError during interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:556
msgid "IOError during interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:566
msgid "Unable to merge file"
msgstr ""

#: ../../client/solo/commands/conf.py:616
msgid "Selected file"
msgstr "Выбранный файл"

#: ../../client/solo/commands/conf.py:638
#, python-format
msgid "Replacing %s with %s"
msgstr ""

#: ../../client/solo/commands/conf.py:645
#, python-format
msgid "Cannot merge %s"
msgstr ""

#: ../../client/solo/commands/conf.py:653
#, python-format
msgid "Deleting %s"
msgstr ""

#: ../../client/solo/commands/conf.py:659
#, python-format
msgid "Cannot remove %s"
msgstr ""

#: ../../client/solo/commands/conf.py:697
msgid "manage package file updates"
msgstr ""

#: ../../client/solo/commands/match.py:81
msgid "match keyword"
msgstr ""

#: ../../client/solo/commands/match.py:94
msgid "match among installed packages only"
msgstr ""

#: ../../client/solo/commands/match.py:98
msgid "match among available packages only"
msgstr ""

#: ../../client/solo/commands/match.py:102
msgid "return all the matches, not just the best"
msgstr ""

#: ../../client/solo/commands/match.py:107
msgid "return matches found in every repository"
msgstr ""

#: ../../client/solo/commands/match.py:113
msgid "print repository information (w/--quiet)"
msgstr "отображает информацию репозитория (с --quiet)"

#: ../../client/solo/commands/match.py:117
msgid "print description too (w/--quiet)"
msgstr "показывает ещё и описание (с --quiet)"

#: ../../client/solo/commands/match.py:121
msgid "print \":<slot>\" next to package names (w/--quiet)"
msgstr ""

#: ../../client/solo/commands/match.py:192
msgid "Matching"
msgstr "Подсчёт"

#: ../../client/solo/commands/match.py:242
msgid "match packages in repositories"
msgstr ""

#: ../../client/solo/commands/install.py:119
#: ../../client/solo/commands/upgrade.py:114
msgid "relax dependencies constraints during calculation"
msgstr ""

#: ../../client/solo/commands/install.py:124
#: ../../client/solo/commands/upgrade.py:119
#: ../../client/solo/commands/download.py:104
msgid "download multiple packages in parallel (max 10)"
msgstr ""

#: ../../client/solo/commands/install.py:183
msgid "These are the packages that would be installed"
msgstr "Здесь перечислены пакеты, которые должны быть установлены"

#: ../../client/solo/commands/install.py:331
msgid "conflicting/substituted"
msgstr "конфликтуют/взаимозаменяют"

#: ../../client/solo/commands/install.py:366
msgid "Packages needing to be installed/updated/downgraded"
msgstr "Пакеты к установке/обновлению/обновлению до более старой версии"

#: ../../client/solo/commands/install.py:371
msgid "Packages needing to be removed"
msgstr "Пакеты к удалению"

#: ../../client/solo/commands/install.py:378
msgid "Packages needing to be installed"
msgstr "Пакеты к установке"

#: ../../client/solo/commands/install.py:385
msgid "Packages needing to be reinstalled"
msgstr "Пакеты к переустановке"

#: ../../client/solo/commands/install.py:392
msgid "Packages needing to be updated"
msgstr "Пакеты к обновлению"

#: ../../client/solo/commands/install.py:399
msgid "Packages needing to be downgraded"
msgstr "Пакеты, которые необходимо обновить до более старой версии"

#: ../../client/solo/commands/install.py:412
msgid "Download size"
msgstr "Размер загрузки"

#: ../../client/solo/commands/install.py:419
msgid "Used disk space"
msgstr "Использованное место на диске"

#: ../../client/solo/commands/install.py:435
msgid "You need at least"
msgstr "Вам нужно совсем немного"

#: ../../client/solo/commands/install.py:437
msgid "of free space"
msgstr "свободного места"

#: ../../client/solo/commands/install.py:451
msgid "You don't have enough space for the installation. Free some space into"
msgstr ""
"У Вас недостаточно места для инсталляции. Освободите немного пространства."

#: ../../client/solo/commands/install.py:549
msgid "Download complete"
msgstr "Загрузка завершена"

#: ../../client/solo/commands/install.py:578
msgid "install"
msgstr "установить"

#: ../../client/solo/commands/install.py:600
msgid "Installation complete"
msgstr "Установка завершена"

#: ../../client/solo/commands/install.py:609
msgid "install or update packages or package files"
msgstr ""

#: ../../client/solo/commands/upgrade.py:166
msgid "Calculating System Updates"
msgstr "Подсчёт системных обновлений"

#: ../../client/solo/commands/upgrade.py:177
msgid "Packages matching update"
msgstr "Обновление пакетного подсчёта"

#: ../../client/solo/commands/upgrade.py:183
msgid "Packages matching not available"
msgstr "Пакетный подсчёт недоступен"

#: ../../client/solo/commands/upgrade.py:189
msgid "Packages matching already up to date"
msgstr "Пакетный подсчёт уже обновлён"

#: ../../client/solo/commands/upgrade.py:212
msgid "Nothing to update"
msgstr "Нечего обновлять"

#: ../../client/solo/commands/upgrade.py:226
msgid ""
"On the system there are packages that are not available anymore in the "
"online repositories"
msgstr ""
"В системе имеются пакеты, которые более недоступны в онлайн-репозиториях"

#: ../../client/solo/commands/upgrade.py:231
msgid ""
"Even if they are usually harmless, it is suggested (after proper "
"verification) to remove them."
msgstr ""

#: ../../client/solo/commands/upgrade.py:249
msgid "Selective"
msgstr ""

#: ../../client/solo/commands/upgrade.py:257
msgid "Would you like to remove them?"
msgstr ""

#: ../../client/solo/commands/upgrade.py:276
msgid "Remove this?"
msgstr ""

#: ../../client/solo/commands/upgrade.py:292
msgid "Nothing to remove"
msgstr "Нечего удалять"

#: ../../client/solo/commands/upgrade.py:321
msgid "There are more updates to install, reloading Entropy"
msgstr ""

#: ../../client/solo/commands/upgrade.py:340
msgid "upgrade the system"
msgstr ""

#: ../../client/solo/commands/rescue.py:89
msgid "check installed packages repository for errors"
msgstr ""

#: ../../client/solo/commands/rescue.py:96
msgid "compact the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:102
msgid ""
"re-generate the installed packages repository using the Source Package "
"Manager"
msgstr ""

#: ../../client/solo/commands/rescue.py:109
msgid "re-generate SPM<->Entropy package UIDs mapping"
msgstr ""

#: ../../client/solo/commands/rescue.py:115
msgid ""
"update Entropy installed packages repository merging Source Package Manager "
"changes"
msgstr ""

#: ../../client/solo/commands/rescue.py:126
msgid "create a backup of the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:132
msgid "restore a backup of the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:171
msgid "Repository error"
msgstr ""

#: ../../client/solo/commands/rescue.py:179
#: ../../client/solo/commands/rescue.py:232
#: ../../client/solo/commands/rescue.py:241
msgid "Sanity Check"
msgstr "проверка работоспособности"

#: ../../client/solo/commands/rescue.py:180
msgid "installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:186
msgid "Scanning..."
msgstr "Сканирование..."

#: ../../client/solo/commands/rescue.py:222
msgid "Error checking package"
msgstr ""

#: ../../client/solo/commands/rescue.py:242
msgid "passed"
msgstr ""

#: ../../client/solo/commands/rescue.py:264
msgid "Compacting the Installed Packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:277
msgid "Compaction complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:306
msgid ""
"the Installed Packages repository will be re-generated using the Source "
"Package Manager"
msgstr ""

#: ../../client/solo/commands/rescue.py:317
msgid "I am not joking, this is quite disruptive"
msgstr ""

#: ../../client/solo/commands/rescue.py:325
msgid "Understood ?"
msgstr "Я ясно излагаю ?"

#: ../../client/solo/commands/rescue.py:329
msgid "Really ?"
msgstr "Правда ?"

#: ../../client/solo/commands/rescue.py:334
msgid "This is your last chance"
msgstr "Это твой последний шанс"

#: ../../client/solo/commands/rescue.py:335
msgid "Ok?"
msgstr "Ok?"

#: ../../client/solo/commands/rescue.py:352
#: ../../client/solo/commands/rescue.py:375
msgid "Cannot read metadata"
msgstr ""

#: ../../client/solo/commands/rescue.py:383
msgid "Creating a backup of the current repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:395
msgid "Cannot backup the repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:412
msgid "Cannot delete old repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:422
msgid "Initializing a new repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:433
msgid "Repository initialized, generating metadata"
msgstr ""

#: ../../client/solo/commands/rescue.py:463
#: ../../client/solo/commands/rescue.py:725
msgid "Invalid package"
msgstr ""

#: ../../client/solo/commands/rescue.py:476
#: ../../client/solo/commands/rescue.py:739
msgid "Metadata generation error"
msgstr ""

#: ../../client/solo/commands/rescue.py:519
msgid "Indexing metadata, please wait..."
msgstr ""

#: ../../client/solo/commands/rescue.py:524
msgid "Repository metadata generation complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:539
msgid "Scanning Source Package Manager repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:553
msgid "Cannot find package"
msgstr ""

#: ../../client/solo/commands/rescue.py:565
msgid "Scanning Entropy repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:636
msgid "These packages were removed"
msgstr ""

#: ../../client/solo/commands/rescue.py:658
#: ../../client/solo/commands/rescue.py:698
msgid "Continue ?"
msgstr ""

#: ../../client/solo/commands/rescue.py:677
msgid "Removal complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:684
msgid "These packages were added"
msgstr ""

#: ../../client/solo/commands/rescue.py:781
msgid "Update complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:793
msgid "Re-generating packages mapping"
msgstr ""

#: ../../client/solo/commands/rescue.py:802
msgid "Packages mapping re-generated"
msgstr ""

#: ../../client/solo/commands/rescue.py:830
msgid "No backups found"
msgstr ""

#: ../../client/solo/commands/rescue.py:852
msgid "Select the repository to restore"
msgstr ""

#: ../../client/solo/commands/rescue.py:859
msgid "Entropy Installed Packages Repository backups"
msgstr ""

#: ../../client/solo/commands/rescue.py:884
msgid "tools to rescue the running system"
msgstr ""

#: ../../client/solo/commands/cleanup.py:97
msgid "remove downloaded packages and clean temp. directories"
msgstr "удалить загруженные пакеты и очистить временные директории"

#: ../../client/solo/commands/unused.py:70
msgid "sort packages by size"
msgstr ""

#: ../../client/solo/commands/unused.py:75
msgid "include packages installed by user"
msgstr ""

#: ../../client/solo/commands/unused.py:114
msgid ""
"Running unused packages test, pay attention, there can be false positives"
msgstr ""

#: ../../client/solo/commands/unused.py:162
msgid "look for unused packages (pay attention)"
msgstr "ищет неиспользуемые пакеты (обратите внимание)"

#: ../../client/solo/commands/download.py:140
msgid "Package download"
msgstr ""

#: ../../client/solo/commands/download.py:172
msgid "download packages, essentially"
msgstr ""

#: ../../client/solo/commands/hop.py:92
msgid "Already on branch"
msgstr "Уже на ветке"

#: ../../client/solo/commands/hop.py:133
msgid "Succesfully switched to branch"
msgstr "Успешно переключено на ветку"

#: ../../client/solo/commands/hop.py:140
msgid "Now run 'equo upgrade' to upgrade your distribution to"
msgstr ""

#: ../../client/solo/commands/hop.py:148
msgid "Unable to switch to branch"
msgstr "Невозможно переключить ветку"

#: ../../client/solo/commands/hop.py:160
msgid "upgrade the System to a new branch"
msgstr ""

#: ../../client/solo/commands/mask.py:53
msgid "only show what would be done"
msgstr "показывать только то, что будет сделано"

#: ../../client/solo/commands/mask.py:125
msgid "These are the packages that would be handled"
msgstr ""

#: ../../client/solo/commands/mask.py:160
msgid "Would you like to continue?"
msgstr ""

#: ../../client/solo/commands/mask.py:174
msgid "action not executed"
msgstr ""

#: ../../client/solo/commands/mask.py:222
msgid "mask one or more packages"
msgstr ""

#: ../../client/solo/commands/mask.py:229
msgid "unmask one or more packages"
msgstr ""

#: ../../magneto/src/magneto/kde/interfaces.py:74
msgid "Magneto Entropy Updates Applet"
msgstr "Апплет обновлений Entropy Magneto"

#: ../../magneto/src/magneto/kde/interfaces.py:229
#: ../../magneto/src/magneto/gtk/interfaces.py:196
msgid "Your system currently has an outdated version of"
msgstr "На Вашей системе в данный момент имеется устаревшая версия"

#: ../../magneto/src/magneto/kde/interfaces.py:230
#: ../../magneto/src/magneto/gtk/interfaces.py:197
msgid "installed"
msgstr "установлено"

#: ../../magneto/src/magneto/kde/interfaces.py:231
#: ../../magneto/src/magneto/gtk/interfaces.py:198
msgid "the latest available version is"
msgstr "наиболее актуальная версия -"

#: ../../magneto/src/magneto/kde/interfaces.py:233
#: ../../magneto/src/magneto/gtk/interfaces.py:200
msgid ""
"It is recommended that you upgrade to the latest before updating any other "
"packages"
msgstr ""
"Рекомендуется обновить до последней версии перед обновлением каких-либо "
"других пакетов"

#: ../../magneto/src/magneto/kde/components.py:83
msgid "Close"
msgstr "Закрыть"

#: ../../magneto/src/magneto/kde/components.py:84
msgid "Launch Application Browser"
msgstr ""

#: ../../magneto/src/magneto/kde/components.py:94
msgid "Application updates"
msgstr "Обновления приложений"

#: ../../magneto/src/magneto/gtk/interfaces.py:133
msgid "System Updates"
msgstr ""

#: ../../magneto/src/magneto/gtk/components.py:88
msgid "Application"
msgstr "Приложения"

#: ../../magneto/src/magneto/gtk/components.py:90
msgid "Latest version"
msgstr "Последняя версия"

#: ../../magneto/src/magneto/gtk/components.py:131
msgid "Critical Information"
msgstr "Критическая информация"

#: ../../magneto/src/magneto/core/interfaces.py:162
msgid "_Disable Notification Applet"
msgstr "_Деактивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:163
msgid "Disable Notification Applet"
msgstr "_Деактивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:164
msgid "_Enable Notification Applet"
msgstr "_Aктивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:165
msgid "Enable Notification Applet"
msgstr "Активировать пакет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:166
msgid "_Check for updates"
msgstr "_Проверить на обновления"

#: ../../magneto/src/magneto/core/interfaces.py:167
msgid "Check for updates"
msgstr "Проверить на обновления"

#: ../../magneto/src/magneto/core/interfaces.py:168
msgid "_Launch Package Manager"
msgstr "_Запустить пакетный менеджер"

#: ../../magneto/src/magneto/core/interfaces.py:169
msgid "Launch Package Manager"
msgstr "Запустить пакетный менеджер"

#: ../../magneto/src/magneto/core/interfaces.py:170
msgid "_Packages Website"
msgstr "_Веб-страница пакетов"

#: ../../magneto/src/magneto/core/interfaces.py:171
msgid "Use Packages web interface"
msgstr "Использовать веб-интерфейс пакетов"

#: ../../magneto/src/magneto/core/interfaces.py:172
msgid "_Sabayon Linux Website"
msgstr "Веб-_сайт Sabayon Linux"

#: ../../magneto/src/magneto/core/interfaces.py:173
msgid "Launch Sabayon Linux Website"
msgstr "Открыть сайт Sabayon Linux"

#: ../../magneto/src/magneto/core/interfaces.py:175
msgid "_Exit"
msgstr "_Выход"

#: ../../magneto/src/magneto/core/interfaces.py:307
msgid "Cannot monitor Sabayon updates"
msgstr "Невозможно отобразить обновления Sabayon"

#: ../../magneto/src/magneto/core/interfaces.py:309
msgid "Entropy DBus service not available"
msgstr "Служба DBus Entropy недоступна"

#: ../../magneto/src/magneto/core/interfaces.py:310
msgid "unable to communicate with the updates service"
msgstr "невозможно соединиться со службой обновлений"

#: ../../magneto/src/magneto/core/interfaces.py:318
msgid "Sabayon updates service loaded"
msgstr "Служба обновлений Sabayon загружена"

#: ../../magneto/src/magneto/core/interfaces.py:320
msgid "Entropy DBus service loaded"
msgstr "Служба DBus Entropy загружена"

#: ../../magneto/src/magneto/core/interfaces.py:321
msgid "your Sabayon will notify you once updates are available"
msgstr "Ваш Sabayon уведомит Вас, когда обновления будут доступны"

#: ../../magneto/src/magneto/core/interfaces.py:334
#, python-format
msgid "There is %s update available"
msgid_plural "There are %s updates available"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../magneto/src/magneto/core/interfaces.py:340
msgid "Sabayon updates available"
msgstr "Обновления Sabayon доступны"

#: ../../magneto/src/magneto/core/interfaces.py:341
#, python-format
msgid "There is <b>%s</b> update available"
msgid_plural "There are <b>%s</b> updates available"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#. all fine, no updates
#: ../../magneto/src/magneto/core/interfaces.py:349
#: ../../magneto/src/magneto/core/interfaces.py:351
msgid "Your Sabayon is up-to-date"
msgstr "Ваш Sabayon уже обновлен до последней версии"

#: ../../magneto/src/magneto/core/interfaces.py:352
msgid "No updates available at this time, cool!"
msgstr "На текущий момент обновлений не обнаружено, отлично!"

#. all fine, no updates
#: ../../magneto/src/magneto/core/interfaces.py:363
msgid "Repositories are being updated"
msgstr "Репозитории обновляются"

#: ../../magneto/src/magneto/core/interfaces.py:364
msgid "Sabayon repositories status"
msgstr "Статус репозиториев Sabayon"

#: ../../magneto/src/magneto/core/interfaces.py:365
msgid "Repositories are being updated automatically"
msgstr "Репозитории обновляются автоматически"

#: ../../magneto/src/magneto/core/interfaces.py:459
msgid "Updates Notification Applet Disabled"
msgstr "Апплет уведомления об обновлениях отключён"

#: ../../magneto/src/magneto/core/interfaces.py:469
msgid "Updates Notification Applet Enabled"
msgstr "Апплет уведомления об обновлениях включён"

#: ../../services/kernel-switcher:139
msgid "has been installed."
msgstr ""

#: ../../services/kernel-switcher:141
msgid "some external drivers cannot work across multiple kernels."
msgstr ""

#: ../../services/kernel-switcher:142
msgid "Please reboot your computer now !"
msgstr ""

#: ../../services/kernel-switcher:164
msgid "Package does not exist"
msgstr ""

#: ../../services/kernel-switcher:171
msgid "Not a kernel"
msgstr ""

#: ../../services/kernel-switcher:203
msgid "Cannot find your currently running kernel."
msgstr ""

#: ../../services/kernel-switcher:205
msgid "Try without --from-running."
msgstr ""

#: ../../services/kernel-switcher:275
msgid "No kernel packages found"
msgstr ""

#: ../../services/kernel-switcher:296
msgid "Sabayon Kernel Switcher"
msgstr ""

#: ../../services/kernel-switcher:303
msgid "quiet mode"
msgstr ""

#: ../../services/kernel-switcher:306
msgid "verbose mode"
msgstr ""

#: ../../services/kernel-switcher:310
msgid "ask confirmation"
msgstr ""

#: ../../services/kernel-switcher:320
msgid "install a new or just another kernel"
msgstr ""

#: ../../services/kernel-switcher:325
msgid "use 'uname -r' to determine the running kernel"
msgstr ""

#: ../../services/kernel-switcher:328
msgid "the new kernel package dependency name"
msgstr ""

#: ../../services/kernel-switcher:332
msgid "list kernels"
msgstr ""

#: ../../magneto/src/magneto/gtk/magneto.glade.h:1
msgid "Magneto Application Updates Notifier"
msgstr "Уведомитель об обновлениях приложений Magneto"

#: ../../magneto/src/magneto/gtk/magneto.glade.h:2
msgid "Available Updates"
msgstr "Доступные обновления"

#: ../../magneto/src/magneto/gtk/magneto.glade.h:3
msgid "_Load Package Manager"
msgstr "_Загрузить пакетный менеджер"
